### Preprints


Jahn D, Knapstein PR, Otto E, Köhli P, Sevecke J, Graef F, Graffmann C, Fuchs M, Jiang S, Rickert M, Erdmann C, Appelt J, Revend L, Küttner Q, Witte J,
Rahmani A, Duda G, Xie W, Donat A, Schinke T, **Ivanov  A**, Tchouto MN, **Beule  D**, Frosch K, Baranowsky A, Tsitsilonis S, Keller J (2023). “Increased
beta2-adrenergic signaling is a targetable stimulus essential for bone healing by promoting callus neovascularization.” _bioRxiv_, 2023-07.
doi:doi.org/10.1101/2023.07.14.548550  [☞ Link](https://www.biorxiv.org/content/10.1101/2023.07.14.548550v1.abstract)

Boeddrich A, Haenig C, Neuendorf N, **Blanc  E**, **Ivanov  A**, Kirchner M, Schleumann P, Bayraktaroglu I, Richter M, Molenda CM, Sporbert A, Zenkner M, Schnoegl
S, Suenkel C, Schneider L, Rybak-Wolf A, Kochnowsky B, Byrne LM, Wild EJ, Nielsen JE, Dittmar G, Peters O, **Beule  D**, Wanker EE (2023). “A proteomics
analysis of 5xFAD mouse brain regions reveals the lysosome-associated protein Arl8b as a candidate biomarker for Alzheimer's disease.” _bioRxiv_,
2023-01. doi:10.1101/2023.01.16.523715  [☞ Link](https://www.biorxiv.org/content/10.1101/2023.01.16.523715v1.abstract)

Ermshaus A, Piechotta M, Ruter G, Keilholz U, Leser U, **Benary  M** (2023). “preon: Fast and accurate entity normalization for drug names and cancer types
in precision oncology.” _bioRxiv_, 2023-05. doi:10.1101/2023.05.22.540912 

Graff P, Wilzopolski J, Voss A, Blimkie TM, **Weiner  J3**, Kershaw O, Panwar P, Hackett T, Brömme D, Loyal L, Thiel A, **Beule  D**, Hancock RE, Gruber AD,
Bäumer W, Hedtrich S (2022). “Extracellular matrix remodeling in atopic dermatitis harnesses the onset of an asthmatic phenotype and is a potential
contributor to the atopic march.” _medRxiv_. doi:10.1101/2022.01.17.22269397  [☞ Link](https://www.medrxiv.org/content/10.1101/2022.01.17.22269397.abstract)

Yerinde C, Keye J, Durlanik S, Freise I, Nowak F, Hsiao H, Letizia M, Schlickeiser S, **Obermayer  B**, Huck A, Friedrich M, Wu H, Kunkel D, Kühl AA, Bauer
S, Thiel A, Siegmund B, Glauben R, Weidinger C (2022). “HDAC7 controls anti-viral and anti-tumor immunity by CD8+ T cells.” _bioRxiv_.
doi:10.1101/2022.09.18.508452  [☞ Link](https://www.biorxiv.org/content/early/2022/09/19/2022.09.18.508452)

Winkler W, Díaz C, **Blanc  E**, Napieczynska H (2021). “Preclinical models of human multiple myeloma subgroups.” _bioRxiv_. doi:10.1101/2021.08.28.458010
 [☞ Link](https://www.biorxiv.org/content/10.1101/2021.08.28.458010.abstract)

Göppel T, **Obermayer  B**, Chen I, Gerland U (2021). “A kinetic error filtering mechanism for enzyme-free copying of nucleic acid sequences.” _bioRxiv_.
doi:10.1101/2021.08.06.455386  [☞ Link](https://www.biorxiv.org/content/10.1101/2021.08.06.455386.abstract)


### 2023


Lechner L, Opitz R, Silver MJ, Krabusch PM, Prentice AM, Field MS, Stachelscheid H, Leitão E, Schröder C, Vallone VF, Horsthemke B, Jöckel K, Schmidt B,
Nöthen MM, Hoffmann P, Herms S, Kleyn PW, Megges M, Blume-Peytavi U, Weiss K, Mai K, Blankenstein O, **Obermayer  B**, Wiegand S, Kühnen PKP (2023).
“Early-set POMC methylation variability is accompanied by increased risk for obesity and is addressable by MC4R agonist treatment.” _Science
Translational Medicine_, *15*(705), eadg1659. doi:10.1126/scitranslmed.adg1659  [☞ Link](https://www.science.org/doi/abs/10.1126/scitranslmed.adg1659)

**Obermayer  B**, Keilholz L, Conrad T, Frentsch M, Blau I, Vuong L, Lesch S, Movasshagi K, Tietze-Stolley C, Loyal L, Henze L, Penack O, Stervbo U, Babel N,
Haas S, **Beule  D**, Bullinger L, Wittenbecher F, Na I (2023). “Single-cell clonal tracking of persistent T-cells in allogeneic hematopoietic stem cell
transplantation.” _Frontiers in Immunology_, *14*. ISSN 1664-3224, doi:10.3389/fimmu.2023.1114368  [☞ Link](https://www.frontiersin.org/articles/10.3389/fimmu.2023.1114368)

Gressier E, Schulte-Schrepping J, Petrov L, Brumhard S, Stubbemann P, Hiller A, **Obermayer  B**, Spitzer J, Kostevc T, Whitney PG, Bachem A, Odainic A, van
de Sandt C, Nguyen THO, Ashhurst T, Wilson K, Oates CVL, Gearing LJ, Meischel T, Hochheiser K, Greyer M, Clarke M, Kreutzenbeck M, Gabriel SS,
Kastenmüller W, Kurts C, Londrigan SL, Kallies A, Kedzierska K, Hertzog PJ, Latz E, Chen YE, Radford KJ, Chopin M, Schroeder J, Kurth F, Gebhardt T,
Sander LE, Sawitzki B, Schultze JL, Schmidt SV, Bedoui S (2023). “CD4+ T cell calibration of antigen-presenting cells optimizes antiviral CD8+ T cell
immunity.” _Nature Immunology_, *24*(6), 979-990. ISSN 1529-2916, doi:10.1038/s41590-023-01517-x  [☞ Link](https://doi.org/10.1038/s41590-023-01517-x)

Loedige I, Baranovskii A, Mendonsa S, Dantsuji S, Popitsch N, Breimann L, Zerna N, Cherepanov V, **Milek  M**, Ameres S, Chekulaeva M (2023). “mRNA stability
and m6A are major determinants of subcellular mRNA localization in neurons.” _Molecular Cell_, *83*(15), 2709-2725.e10. ISSN 1097-2765,
doi:10.1016/j.molcel.2023.06.021  [☞ Link](https://www.sciencedirect.com/science/article/pii/S1097276523004689)

de Bortoli T, **Benary  M**, Horak P, Lamping M, Stintzing S, Tinhofer I, Leyvraz S, Schäfer R, Klauschen F, Keller U, Rieke DT, Jelas I (2023). “Tumour
mutational burden and survival with molecularly matched therapy.” _European Journal of Cancer_, *190*, 112925. doi:10.1016/j.ejca.2023.05.013
 [☞ Link](https://www.sciencedirect.com/science/article/abs/pii/S0959804923002538)

Stellmach C, Sass J, Auber B, Boeker M, Wienker T, Heidel AJ, **Benary  M**, Schumacher S, Ossowski S, Klauschen F, Möller Y, Schmutzler R, Ustjanzew A,
Werner P, Tomczak A, Hölter T, Thun S (2023). “Creation of a structured molecular genomics report for Germany as a local adaption of HL7’s Genomic
Reporting Implementation Guide.” _Journal of the American Medical Informatics Association_. ISSN 1527-974X, doi:10.1093/jamia/ocad061
 [☞ Link](https://doi.org/10.1093/jamia/ocad061)

Winkler W, Díaz CF, **Blanc  E**, Napieczynska H, Langner P, Werner M, Walter B, Wollert-Wulf B, Yasuda T, Heuser A, **Beule  D**, Mathas S, Anagnostopoulos I,
Rosenwald A, Rajewsky K, Janz M (2023). “Mouse models of human multiple myeloma subgroups.” _Proceedings of the National Academy of Sciences of the
United States of America_, *120*, e2219439120. ISSN 1091-6490, doi:10.1073/pnas.2219439120  [☞ Link](https://doi.org/10.1073/pnas.2219439120)

Spranger L, **Weiner  J**, Bredow J, Zeitz U, Grittner U, Boschmann M, Dickmann S, Stobaus N, Schwartzenberg JJ, Brachs M, Spranger J, Mai K (2023). “Thrifty
energy phenotype predicts weight regain in postmenopausal women with overweight or obesity and is related to FGFR1 signaling.” _Clinical Nutrition_,
*42*, 559-567. doi:10.1016/j.clnu.2023.02.020  [☞ Link](https://www.sciencedirect.com/science/article/abs/pii/S0261561423000560)

Freitag K, Eede P, **Ivanov  A**, Sterczyk N, Schneeberger S, Borodina T, Sauer S, **Beule  D**, Heppner F (2023). “Diverse but unique astrocytic phenotypes
during embryonic stem cell differentiation, culturing and development.” _Commun Biol_, *6*, 40. doi:10.1038/s42003-023-04410-3
 [☞ Link](https://doi.org/10.1038/s42003-023-04410-3)

**Nieminen  M**, **Stolpe  O**, **Kuhring  M**, **Weiner  J**, **Pett  P**, **Beule  D**, **Holtgrewe  M** (2023). “SODAR: managing multiomics study data and metadata.” _Gigascience_,
*12*, giad052. doi:10.1093/gigascience/giad052  [☞ Link](https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giad052/7232111)

Müller R, Freitag-Wolf S, **Weiner  J3**, Chopra A, Top T, Dommisch H, Schaefer A (2023). “Case-only design identifies interactions of genetic risk variants
at SIGLEC5 and PLG with the lncRNA CTD-2353F22.1 implying the importance of periodontal wound healing for disease aetiology.” _J Clin Periodontol_,
*50*, 90-101. doi:10.1111/jcpe.13712  [☞ Link](https://doi.org/10.1111/jcpe.13712)

Mothes R, Pascual-Reguant A, Koehler R, Liebeskind J, Liebheit A, Bauherr S, Philipsen L, Dittmayer C, Laue M, von Manitius R, Elezkurtaj S, Durek P,
Heinrich F, Heinz GA, Guerra GM, **Obermayer  B**, Meinhardt J, Ihlow J, Radke J, Heppner FL, Enghard P, Stockmann H, Aschman T, Schneider J, Corman VM,
Sander LE, Mashreghi M, Conrad T, Hocke AC, Niesner RA, Radbruch H, Hauser AE (2023). “Distinct tissue niches direct lung immunopathology via CCL18 and
CCL21 in severe COVID-19.” _Nature Communications_, *14*(1), 791. ISSN 2041-1723, doi:10.1038/s41467-023-36333-2
 [☞ Link](https://doi.org/10.1038/s41467-023-36333-2)

Rieke DT, Schröder S, Schafhausen P, **Blanc  E**, **Zuljan  E**, von der Emde B, **Beule  D**, Keller U, Keilholz U, Klinghammer K (2023). “Targeted treatment in a
case series of AR+, HRAS/PIK3CA co-mutation salivary duct carcinoma.” _Frontiers in Oncology_, *13*. ISSN 2234-943X, doi:10.3389/fonc.2023.1107134



### 2022


Baumgardt M, Hülsemann M, Löwa A, Fatykhova D, Hoffmann K, Kessler M, Mieth M, Hellwig K, Frey D, Langenhagen A, Voss A, **Obermayer  B**, Wyler E, Dökel S,
Gruber AD, Tölch U, Hippenstiel S, Hocke AC, Hönzke K (2022). “State-of-the-art analytical methods of viral infections in human lung organoids.” _PLOS
ONE_, *17*(12), 1-8. doi:10.1371/journal.pone.0276115  [☞ Link](https://doi.org/10.1371/journal.pone.0276115)

Pennitz P, Kirsten H, Friedrich VD, Wyler E, Goekeri C, **Obermayer  B**, Heinz GA, Mashreghi M, B"uttner M, Trimpert J, Landthaler M, Suttorp N, Hocke AC,
Hippenstiel S, T"onnies M, Scholz M, Kuebler WM, Witzenrath M, Hoenzke K, Nouailles G (2022). “A pulmonologist's guide to perform and analyse
cross-species single lung cell transcriptomics.” _European Respiratory Review_, *31*(165). ISSN 0905-9180, doi:10.1183/16000617.0056-2022
 [☞ Link](https://err.ersjournals.com/content/31/165/220056)

Bertrams W, Hönzke K, **Obermayer  B**, Tönnies M, Bauer TT, Schneider P, Neudecker J, Rückert JC, Stiewe T, Nist A, Eggeling S, Suttorp N, Wolff T,
Hippenstiel S, Schmeck B, Hocke AC (2022). “Transcriptomic comparison of primary human lung cells with lung tissue samples and the human A549 lung cell
line highlights cell type specific responses during infections with influenza A virus.” _Scientific Reports_, *12*(1), 20608. ISSN 2045-2322,
doi:10.1038/s41598-022-24792-4  [☞ Link](https://doi.org/10.1038/s41598-022-24792-4)

Jarick K, Topczewska P, Jakob M, Yano H, Arifuzzaman M, Gao X, Boulekou S, Stokic-Trtica V, Leclère P, Preußer A, Rompe Z, Stamm A, Tsou A, Chu C,
Heinrich F, Guerra G, Durek P, **Ivanov  A**, **Beule  D**, Helfrich S, Duerr C, Kühl A, Stehle C, Romagnani C, Mashreghi M, Diefenbach A, Artis D, Klose C
(2022). “Non-redundant functions of group 2 innate lymphoid cells.” _Nature_, *611*(7937), 794-800. doi:10.1038/s41586-022-05395-5
 [☞ Link](https://doi.org/10.1038/s41586-022-05395-5)

**Ivanov  A**, Mattei D, Radscheit K, Compagnion A, **Pett  J**, Herzel H, Paolicelli R, Piwecka M, Meyer U, **Beule  D** (2022). “Analyses of circRNA Expression
throughout the Light-Dark Cycle Reveal a Strong Regulation of Cdr1as, Associated with Light Entrainment in the SCN.” _Int J Mol Sci_, *23*.
doi:10.3390/ijms232012347  [☞ Link](https://doi.org/10.3390/ijms232012347)

Wirges A, Bunse M, Joedicke J, **Blanc  E**, Gudipati V, Moles M, Shiku H, **Beule  D**, Huppa J, Höpken U, Rehm A (2022). “EBAG9 silencing exerts an immune
checkpoint function without aggravating adverse effects.” _Mol Ther_, *30*, 3358-3378. doi:10.1016/j.ymthe.2022.07.009
 [☞ Link](https://doi.org/10.1016/j.ymthe.2022.07.009)

Hönzke K, **Obermayer  B**, Mache C, Fathykova D, Kessler M, Dökel S, Wyler E, Baumgardt M, Löwa A, Hoffmann K, Graff P, Schulze J, Mieth M, Hellwig K, Demir
Z, Biere B, Brunotte L, Mecate-Zambrano A, Bushe J, Dohmen M, Hinze C, Elezkurtaj S, Tönnies M, Bauer T, Eggeling S, Tran H, Schneider P, Neudecker J,
Rückert J, Schmidt-Ott K, Busch J, Klauschen F, Horst D, Radbruch H, Radke J, Heppner F, Corman V, Niemeyer D, Müller M, Goffinet C, Mothes R,
Pascual-Reguant A, Hauser A, **Beule  D**, Landthaler M, Ludwig S, Suttorp N, Witzenrath M, Gruber A, Drosten C, Sander L, Wolff T, Hippenstiel S, Hocke A
(2022). “Human lungs show limited permissiveness for SARS-CoV-2 due to scarce ACE2 levels but virus-induced expansion of inflammatory macrophages.” _Eur
Respir J_, *60*. doi:10.1183/13993003.02725-2021  [☞ Link](https://doi.org/10.1183/13993003.02725-2021)

Freitag K, Sterczyk N, Wendlinger S, **Obermayer  B**, Schulz J, Farztdinov V, Mülleder M, Ralser M, Houtman J, Fleck L, Braeuning C, Sansevrino R, Hoffmann
C, Milovanovic D, Sigrist S, Conrad T, **Beule  D**, Heppner F, Jendrach M (2022). “Spermidine reduces neuroinflammation and soluble amyloid beta in an
Alzheimer's disease mouse model.” _J Neuroinflammation_, *19*, 172. doi:10.1186/s12974-022-02534-7  [☞ Link](https://doi.org/10.1186/s12974-022-02534-7)

Schalbetter S, von Arx A, Cruz-Ochoa N, Dawson K, **Ivanov  A**, Mueller F, Lin H, Amport R, Mildenberger W, Mattei D, **Beule  D**, Földy C, Greter M, Notter T,
Meyer U (2022). “Adolescence is a sensitive period for prefrontal microglia to act on cognitive development.” _Sci Adv_, *8*, eabi6672.
doi:10.1126/sciadv.abi6672  [☞ Link](https://doi.org/10.1126/sciadv.abi6672)

Grunert C, Willimsky G, Peuker C, Rhein S, Hansmann L, Blankenstein T, **Blanc  E**, **Beule  D**, Keller U, Pezzutto A, Busse A (2022). “Isolation of
Neoantigen-Specific Human T Cell Receptors from Different Human and Murine Repertoires.” _Cancers (Basel)_, *14*. doi:10.3390/cancers14071842
 [☞ Link](https://doi.org/10.3390/cancers14071842)

Dannebaum R, Suwalski P, Asgharian H, Du ZG, Lin H, **Weiner  J**, **Holtgrewe  M**, Thibeault C, Müller M, Wang X, Karadeniz Z, Saccomanno J, Doehn J, Hübner R,
Hinzmann B, Blüher A, Siemann S, Telman D, Suttorp N, Witzenrath M, Hippenstiel S, Skurk C, Poller W, Sander L, **Beule  D**, Kurth F, Guettouche T,
Landmesser U, Berka J, Luong K, Pa-COVID SG, Rubelt F, Heidecker B (2022). “Highly multiplexed immune repertoire sequencing links multiple lymphocyte
classes with severity of response to COVID-19.” _EClinicalMedicine_, *48*, 101438. doi:10.1016/j.eclinm.2022.101438
 [☞ Link](https://doi.org/10.1016/j.eclinm.2022.101438)

Horak P, Griffith M, Danos A, Pitel B, Madhavan S, Liu X, Chow C, Williams H, Carmody L, Barrow-Laing L, Rieke D, Kreutzfeldt S, Stenzinger A, Tamborero
D, **Benary  M**, Rajagopal P, Ida C, Lesmana H, Satgunaseelan L, Merker J, Tolstorukov M, Campregher P, Warner J, Rao S, Natesan M, Shen H, Venstrom J, Roy
S, Tao K, Kanagal-Shamanna R, Xu X, Ritter D, Pagel K, Krysiak K, Dubuc A, Akkari Y, Li X, Lee J, King I, Raca G, Wagner A, Li M, Plon S, Kulkarni S,
Griffith O, Chakravarty D, Sonkin D (2022). “Standards for the classification of pathogenicity of somatic variants in cancer (oncogenicity): Joint
recommendations of Clinical Genome Resource (ClinGen), Cancer Genomics Consortium (CGC), and Variant Interpretation for Cancer Consortium (VICC).”
_Genet Med_, *24*, 986-998. doi:10.1016/j.gim.2022.01.001  [☞ Link](https://doi.org/10.1016/j.gim.2022.01.001)

**Weiner  J**, **Obermayer  B**, **Beule  D** (2022). “Venn Diagrams May Indicate Erroneous Statistical Reasoning in Transcriptomics.” _Front Genet_, *13*, 818683.
doi:10.3389/fgene.2022.818683  [☞ Link](https://doi.org/10.3389/fgene.2022.818683)

Gloaguen Y, Kirwan J, **Beule  D** (2022). “Deep Learning-Assisted Peak Curation for Large-Scale LC-MS Metabolomics.” _Anal Chem_, *94*, 4930-4937.
doi:10.1021/acs.analchem.1c02220  [☞ Link](https://doi.org/10.1021/acs.analchem.1c02220)

Chopra A, Mueller R, **Weiner  J3**, Rosowski J, Dommisch H, Grohmann E, Schaefer A (2022). “BACH1 Binding Links the Genetic Risk for Severe Periodontitis
with ST8SIA1.” _J Dent Res_, *101*, 93-101. doi:10.1177/00220345211017510  [☞ Link](https://doi.org/10.1177/00220345211017510)

Bischoff P, Trinks A, Wiederspahn J, **Obermayer  B**, **Pett  J**, Jurmeister P, Elsner A, Dziodzio T, Rückert J, Neudecker J, Falk C, **Beule  D**, Sers C, Morkel M,
Horst D, Klauschen F, Blüthgen N (2022). “The single-cell transcriptional landscape of lung carcinoid tumors.” _Int J Cancer_, *150*, 2058-2071.
doi:10.1002/ijc.33995  [☞ Link](https://doi.org/10.1002/ijc.33995)

Georg P, Astaburuaga-García R, Bonaguro L, Brumhard S, Michalick L, Lippert L, Kostevc T, Gäbel C, Schneider M, Streitz M, Demichev V, Gemünd I, Barone
M, Tober-Lau P, Helbig E, Hillus D, Petrov L, Stein J, Dey H, Paclik D, Iwert C, Mülleder M, Aulakh S, Djudjaj S, Bülow R, Mei H, Schulz A, Thiel A,
Hippenstiel S, Saliba A, Eils R, Lehmann I, Mall M, Stricker S, Röhmel J, Corman V, **Beule  D**, Wyler E, Landthaler M, **Obermayer  B**, von Stillfried S, Boor
P, Demir M, Wesselmann H, Suttorp N, Uhrig A, Müller-Redetzky H, Nattermann J, Kuebler W, Meisel C, Ralser M, Schultze J, Aschenbrenner A, Thibeault C,
Kurth F, Sander L, Blüthgen N, Sawitzki B, PA-COVID-19 SG (2022). “Complement activation induces excessive T cell cytotoxicity in severe COVID-19.”
_Cell_, *185*(3), 493-512.e25. ISSN 0092-8674, doi:10.1016/j.cell.2021.12.040  [☞ Link](https://doi.org/10.1016/j.cell.2021.12.040)

Zinnall U, **Milek  M**, Minia I, Vieira-Vieira C, Müller S, Mastrobuoni G, Hazapis O, Del GS, Schwefel D, Bley N, Voigt F, Chao J, Kempa S, Hüttelmaier S,
Selbach M, Landthaler M (2022). “HDLBP binds ER-targeted mRNAs by multivalent interactions to promote protein synthesis of transmembrane and secreted
proteins.” _Nat Commun_, *13*(1), 2727. doi:10.1038/s41467-022-30322-7  [☞ Link](https://doi.org/10.1038/s41467-022-30322-7)

Dörr D, **Obermayer  B**, **Weiner  J**, Zimmermann K, Anania C, Wagner L, Lyras E, Sapozhnikova V, Lara-Astiaso D, Prósper F, Lang R, Lupiáñez D, **Beule  D**, Höpken
U, Leutz A, Mildner A (2022). “C/EBPβ regulates lipid metabolism and Pparg isoform 2 expression in alveolar macrophages.” _Sci Immunol_, *7*(75),
eabj0140. doi:10.1126/sciimmunol.abj0140  [☞ Link](https://doi.org/10.1126/sciimmunol.abj0140)

Kedziora S, **Obermayer  B**, Sugulle M, Herse F, Kräker K, Haase N, Langmia I, Müller D, Staff A, **Beule  D**, Dechend R (2022). “Placental Transcriptome
Profiling in Subtypes of Diabetic Pregnancies Is Strongly Confounded by Fetal Sex.” _Int J Mol Sci_, *23*(23). ISSN 1422-0067, doi:10.3390/ijms232315388
 [☞ Link](https://doi.org/10.3390/ijms232315388)

Pennitz P, Kirsten H, Wyler E, Goekeri C, **Obermayer  B**, Heinz GA, Mashreghi MF, Trimpert J, Landthaler M, Suttorp N, Hocke AC, Hippenstiel S, Tönnies M,
Scholz M, Kuebler WM, Witzenrath M, Hoenzke K, Nouailles G (2022). “Integrational approaches for cross-species analysis of lung pathologies at
single-cell resolution.” _Pneumologie_, *76*(S 01), FV 285. FV 285.

Hoffmann K, **Obermayer  B**, Hönzke K, Fatykhova D, Demir Z, Löwa A, Alves L, Wyler E, Lopez-Rodriguez E, Mieth M, Baumgardt M, Hoppe J, Firsching T,
Tönnies M, Bauer T, Eggeling S, Tran H, Schneider P, Neudecker J, Rückert J, Gruber A, Ochs M, Landthaler M, **Beule  D**, Suttorp N, Hippenstiel S, Hocke A,
Kessler M (2022). “Human alveolar progenitors generate dual lineage bronchioalveolar organoids.” _Commun Biol_, *5*(1), 875. ISSN 2399-3642,
doi:10.1038/s42003-022-03828-5  [☞ Link](https://doi.org/10.1038/s42003-022-03828-5)

Neitzel H, Varon R, Chughtai S, Dartsch J, Dutrannoy-Tönsing V, Nürnberg P, Nürnberg G, Schweiger M, Digweed M, Hildebrand G, Hackmann K, **Holtgrewe  M**,
Sarioglu N, Schulze B, Horn D, Sperling K (2022). “Transmission ratio distortion of mutations in the master regulator of centriole biogenesis PLK4.”
_Hum Genet_, *141*(11), 1785-1794. doi:10.1007/s00439-022-02461-w  [☞ Link](https://doi.org/10.1007/s00439-022-02461-w)

Foddis M, Blumenau S, **Holtgrewe  M**, Paquette K, Westra K, Alonso I, Macario M, Morgadinho A, Velon A, Santo G, Santana I, Mönkäre S, Kuuluvainen L,
Schleutker J, Pöyhönen M, Myllykangas L, Pavlovic A, Kostic V, Dobricic V, Lohmann E, Hanagasi H, Santos M, Guven G, Bilgic B, Bras J, **Beule  D**, Dirnagl
U, Guerreiro R, Sassi C (2022). “TREX1 p.A129fs and p.Y305C variants in a large multi-ethnic cohort of CADASIL-like unrelated patients.”
doi:10.1016/j.neurobiolaging.2022.11.013  [☞ Link](https://doi.org/10.1016/j.neurobiolaging.2022.11.013)

Boschann F, Moreno D, Mensah M, Sczakiel H, Skipalova K, **Holtgrewe  M**, Mundlos S, Fischer-Zirnsak B (2022). “Xq27.1 palindrome mediated interchromosomal
insertion likely causes familial congenital bilateral laryngeal abductor paralysis (Plott syndrome).” _J Hum Genet_, *67*(7), 405-410.
doi:10.1038/s10038-022-01018-z  [☞ Link](https://doi.org/10.1038/s10038-022-01018-z)

Seidel F, Laser K, Klingel K, Dartsch J, Theisen S, Pickardt T, **Holtgrewe  M**, Gärtner A, Berger F, **Beule  D**, Milting H, Schubert S, Klaassen S, Kühnisch J
(2022). “Pathogenic Variants in Cardiomyopathy Disorder Genes Underlie Pediatric Myocarditis-Further Impact of Heterozygous Immune Disorder Gene
Variants?” _J Cardiovasc Dev Dis_, *9*(7), 216. doi:10.3390/jcdd9070216  [☞ Link](https://doi.org/10.3390/jcdd9070216)

May V, Koch L, Fischer-Zirnsak B, Horn D, Gehle P, Kornak U, **Beule  D**, **Holtgrewe  M** (2022). “ClearCNV: CNV calling from NGS panel data in the presence of
ambiguity and noise.” _Bioinformatics_, *38*(16), 3871-3876. doi:10.1093/bioinformatics/btac418  [☞ Link](https://doi.org/10.1093/bioinformatics/btac418)

Schöpflin R, Melo U, Moeinzadeh H, Heller D, Laupert V, Hertzberg J, **Holtgrewe  M**, Alavi N, Klever M, Jungnitsch J, Comak E, Türkmen S, Horn D, Duffourd
Y, Faivre L, Callier P, Sanlaville D, Zuffardi O, Tenconi R, Kurtas N, Giglio S, Prager B, Latos-Bielenska A, Vogel I, Bugge M, Tommerup N, Spielmann M,
Vitobello A, Kalscheuer V, Vingron M, Mundlos S (2022). “Integration of Hi-C with short and long-read genome sequencing reveals the structure of
germline rearranged genomes.” _Nat Commun_, *13*(1), 6470. doi:10.1038/s41467-022-34053-7  [☞ Link](https://doi.org/10.1038/s41467-022-34053-7)

Rieke D, de BT, Horak P, Lamping M, **Benary  M**, Jelas I, Rüter G, Berger J, Zettwitz M, Kagelmann N, Kind A, Fabian F, **Beule  D**, Glimm H, Brors B,
Stenzinger A, Fröhling S, Keilholz U (2022). “Feasibility and outcome of reproducible clinical interpretation of high-dimensional molecular data: a
comparison of two molecular tumor boards.” _BMC Med_, *20*, 367. ISSN 17417015, doi:10.1186/s12916-022-02560-5
 [☞ Link](https://doi.org/10.1186/s12916-022-02560-5)

Hinze C, Kocks C, Leiz J, Karaiskos N, Boltengagen A, Cao S, Skopnik C, Klocke J, Hardenberg J, Stockmann H, Gotthardt I, **Obermayer  B**, Haghverdi L,
Wyler E, Landthaler M, Bachmann S, Hocke A, Corman V, Busch J, Schneider W, Himmerkus N, Bleich M, Eckardt K, Enghard P, Rajewsky N, Schmidt-Ott K
(2022). “Single-cell transcriptomics reveals common epithelial response patterns in human acute kidney injury.” _Genome Med_, *14*, 103.
doi:10.1186/s13073-022-01108-9  [☞ Link](https://doi.org/10.1186/s13073-022-01108-9)

Richter G, Wagner G, Reichenmiller K, Staufenbiel I, Martins O, Löscher B, **Holtgrewe  M**, Jepsen S, Dommisch H, Schaefer A (2022). “Exome Sequencing of 5
Families with Severe Early-Onset Periodontitis.” _J Dent Res_, *101*(2), 151-157. doi:10.1177/00220345211029266
 [☞ Link](https://doi.org/10.1177/00220345211029266)

Coutelier M, **Holtgrewe  M**, Jäger M, Flöttman R, Mensah M, Spielmann M, Krawitz P, Horn D, **Beule  D**, Mundlos S (2022). “Combining callers improves the
detection of copy number variants from whole-genome sequencing.” _Eur J Hum Genet_, *30*(2), 178-186. doi:10.1038/s41431-021-00983-x
 [☞ Link](https://doi.org/10.1038/s41431-021-00983-x)

Vogt G, Verheyen S, Schwartzmann S, Ehmke N, Potratz C, Schwerin-Nagel A, Plecko B, **Holtgrewe  M**, Seelow D, Blatterer J, Speicher M, Kornak U, Horn D,
Mundlos S, Fischer-Zirnsak B, Boschann F (2022). “Biallelic truncating variants in ATP9A cause a novel neurodevelopmental disorder involving postnatal
microcephaly and failure to thrive.” _J Med Genet_, *59*(7), 662-668. doi:10.1136/jmedgenet-2021-107843  [☞ Link](https://doi.org/10.1136/jmedgenet-2021-107843)


### 2021


Schinke C, Fernandez VV, **Ivanov  A**, Peng Y, Körtvelyessy P, Nolte L, Huehnchen P, **Beule  D**, Stachelscheid H, Boehmerle W, Endres M (2021). “Dataset for:
Modeling chemotherapy induced neurotoxicity with human induced pluripotent stem cell (iPSC)-derived sensory neurons.” _Data Brief_, *38*, 107320.
doi:10.1016/j.dib.2021.107320  [☞ Link](https://doi.org/10.1016/j.dib.2021.107320)

**Weiner  J**, Suwalski P, **Holtgrewe  M**, Rakitko A, Thibeault C, Müller M, Patriki D, Quedenau C, Krüger U, Ilinsky V, Popov I, Balnis J, Jaitovich A, Helbig
E, Lippert L, Stubbemann P, Real L, Macías J, Pineda J, Fernandez-Fuertes M, Wang X, Karadeniz Z, Saccomanno J, Doehn J, Hübner R, Hinzmann B, Salvo M,
Blueher A, Siemann S, Jurisic S, Beer J, Rutishauser J, Wiggli B, Schmid H, Danninger K, Binder R, Corman V, Mühlemann B, Arjun AR, Fragiadakis G, Mick
E, Comet C, Calfee C, Erle D, Hendrickson C, Kangelaris K, Krummel M, Woodruff P, Langelier C, Venkataramani U, García F, Zyla J, Drosten C, Alice B,
Jones T, Suttorp N, Witzenrath M, Hippenstiel S, Zemojtel T, Skurk C, Poller W, Borodina T, Pa-Covid S, Ripke S, Sander L, **Beule  D**, Landmesser U,
Guettouche T, Kurth F, Heidecker B (2021). “Increased risk of severe clinical course of COVID-19 in carriers of HLA-C\*04:01.” _EClinicalMedicine_,
*40*, 101099. doi:10.1016/j.eclinm.2021.101099  [☞ Link](https://doi.org/10.1016/j.eclinm.2021.101099)

Domaszewska T, Zyla J, Otto R, Kaufmann S, **Weiner  J** (2021). “Gene Set Enrichment Analysis Reveals Individual Variability in Host Responses in
Tuberculosis Patients.” _Front Immunol_, *12*, 694680. doi:10.3389/fimmu.2021.694680  [☞ Link](https://doi.org/10.3389/fimmu.2021.694680)

Nouailles G, Wyler E, Pennitz P, Postmus D, Vladimirova D, Kazmierski J, Pott F, Dietert K, Muelleder M, Farztdinov V, **Obermayer  B**, Wienhold S,
Andreotti S, Hoefler T, Sawitzki B, Drosten C, Sander LE, Suttorp N, Ralser M, **Beule  D**, Gruber AD, Goffinet C, Landthaler M, Trimpert J, Witzenrath M
(2021). “Single-cell-sequencing in SARS-COV-2-infected hamsters sheds light on endothelial cell involvement in COVID-19.” _European Respiratory
Journal_. doi:10.1183/13993003.congress-2021.PA2355  [☞ Link](https://erj.ersjournals.com/content/58/suppl_65/PA2355.abstract)

Bischoff P, Trinks A, **Obermayer  B**, **Pett  J**, Wiederspahn J, Uhlitz F, Liang X, Lehmann A, Jurmeister P, Elsner A, Dziodzio T, Rückert J, Neudecker J, Falk
C, **Beule  D**, Sers C, Morkel M, Horst D, Blüthgen N, Klauschen F (2021). “Single-cell RNA sequencing reveals distinct tumor microenvironmental patterns in
lung adenocarcinoma.” _Oncogene_, *40*, 6748-6758. doi:10.1038/s41388-021-02054-3  [☞ Link](https://doi.org/10.1038/s41388-021-02054-3)

Uhlitz F, Bischoff P, Peidli S, Sieber A, Trinks A, Lüthen M, **Obermayer  B**, **Blanc  E**, Ruchiy Y, Sell T, Mamlouk S, Arsie R, Wei T, Klotz-Noack K, Schwarz
R, Sawitzki B, Kamphues C, **Beule  D**, Landthaler M, Sers C, Horst D, Blüthgen N, Morkel M (2021). “Mitogen-activated protein kinase activity drives cell
trajectories in colorectal cancer.” _EMBO Mol Med_, *13*, e14123. doi:10.15252/emmm.202114123  [☞ Link](https://doi.org/10.15252/emmm.202114123)

Nouailles G, Wyler E, Pennitz P, Postmus D, Vladimirova D, Kazmierski J, Pott F, Dietert K, Muelleder M, Farztdinov V, **Obermayer  B**, Wienhold S,
Andreotti S, Hoefler T, Sawitzki B, Drosten C, Sander L, Suttorp N, Ralser M, **Beule  D**, Gruber A, Goffinet C, Landthaler M, Trimpert J, Witzenrath M
(2021). “Temporal omics analysis in Syrian hamsters unravel cellular effector responses to moderate COVID-19.” _Nat Commun_, *12*, 4869.
doi:10.1038/s41467-021-25030-7  [☞ Link](https://doi.org/10.1038/s41467-021-25030-7)

Paterson S, Kar S, Ung S, Gardener Z, Bergstrom E, Ascough S, Kalyan M, Zyla J, Maertzdorf J, Mollenkopf H, **Weiner  J**, Jozwik A, Jarvis H, Jha A,
Nicholson B, Veldman T, Woods C, Mallia P, Kon O, Kaufmann S, Openshaw P, Chiu C (2021). “Innate-like Gene Expression of Lung-Resident Memory CD8(+) T
Cells during Experimental Human Influenza: A Clinical Study.” _Am J Respir Crit Care Med_, *204*, 826-841. doi:10.1164/rccm.202103-0620OC
 [☞ Link](https://doi.org/10.1164/rccm.202103-0620OC)

Rajasekaran S, Siddiqui J, Rakijas J, Nicolay B, Lin C, Khan E, Patel R, Morris R, Wyler E, Boukhali M, Balasubramanyam J, Ranjith KR, van Rechem C,
Vogel C, Elchuri S, Landthaler M, **Obermayer  B**, Haas W, Dyson N, Miles W (2021). “Integrated multi-omics analysis of RB-loss identifies widespread
cellular programming and synthetic weaknesses.” _Commun Biol_, *4*, 977. doi:10.1038/s42003-021-02495-2  [☞ Link](https://doi.org/10.1038/s42003-021-02495-2)

Schinke C, Fernandez VV, **Ivanov  A**, Peng Y, Körtvelyessy P, Nolte L, Huehnchen P, **Beule  D**, Stachelscheid H, Boehmerle W, Endres M (2021). “Modeling
chemotherapy induced neurotoxicity with human induced pluripotent stem cell (iPSC) -derived sensory neurons.” _Neurobiol Dis_, *155*, 105391.
doi:10.1016/j.nbd.2021.105391  [☞ Link](https://doi.org/10.1016/j.nbd.2021.105391)

Pei G, Zyla J, He L, Moura-Alves P, Steinle H, Saikali P, Lozza L, Nieuwenhuizen N, **Weiner  J**, Mollenkopf H, Ellwanger K, Arnold C, Duan M, Dagil Y,
Pashenkov M, Boneca I, Kufer T, Dorhoi A, Kaufmann S (2021). “Cellular stress promotes NOD1/2-dependent inflammation via the endogenous metabolite
sphingosine-1-phosphate.” _EMBO J_, *40*, e106272. doi:10.15252/embj.2020106272  [☞ Link](https://doi.org/10.15252/embj.2020106272)

Herda S, Heimann A, **Obermayer  B**, Ciraolo E, Althoff S, Ruß J, Grunert C, Busse A, Bullinger L, Pezzutto A, Blankenstein T, **Beule  D**, Na I (2021).
“Long-term in vitro expansion ensures increased yield of central memory T cells as perspective for manufacturing challenges.” _Int J Cancer_, *148*,
3097-3110. doi:10.1002/ijc.33523  [☞ Link](https://doi.org/10.1002/ijc.33523)

Kraushar M, Krupp F, Harnett D, Turko P, Ambrozkiewicz M, Sprink T, Imami K, Günnigmann M, Zinnall U, Vieira-Vieira C, Schaub T, Münster-Wandowski A,
Bürger J, Borisova E, Yamamoto H, Rasin M, Ohler U, **Beule  D**, Mielke T, Tarabykin V, Landthaler M, Kramer G, Vida I, Selbach M, Spahn C (2021). “Protein
Synthesis in the Developing Neocortex at Near-Atomic Resolution Reveals Ebp1-Mediated Neuronal Proteostasis at the 60S Tunnel Exit.” _Mol Cell_, *81*,
304-322.e16. doi:10.1016/j.molcel.2020.11.037  [☞ Link](https://doi.org/10.1016/j.molcel.2020.11.037)

Heyckendorf J, Marwitz S, Reimann M, Avsar K, DiNardo A, Günther G, Hoelscher M, Ibraim E, Kalsdorf B, Kaufmann S, Kontsevaya I, van Leth F, Mandalakas
A, Maurer F, Müller M, Nitschkowski D, Olaru I, Popa C, Rachow A, Rolling T, Rybniker J, Salzer H, Sanchez-Carballo P, Schuhmann M, Schaub D, Spinu V,
Suárez I, Terhalle E, Unnewehr M, **Weiner  J3**, Goldmann T, Lange C (2021). “Prediction of anti-tuberculosis treatment duration based on a 22-gene
transcriptomic model.” _Eur Respir J_, *58*. doi:10.1183/13993003.03492-2020  [☞ Link](https://doi.org/10.1183/13993003.03492-2020)

Bao X, **Weiner  J3**, Meckes O, Dommisch H, Schaefer A (2021). “Entamoeba gingivalis Exerts Severe Pathogenic Effects on the Oral Mucosa.” _J Dent Res_,
*100*, 771-776. doi:10.1177/00220345211004498  [☞ Link](https://doi.org/10.1177/00220345211004498)

Eckold C, Kumar V, **Weiner  J**, Alisjahbana B, Riza A, Ronacher K, Coronel J, Kerry-Barnard S, Malherbe S, Kleynhans L, Stanley K, Ruslami R, Ioana M,
Ugarte-Gil C, Walzl G, van Crevel R, Wijmenga C, Critchley J, Dockrell H, Cliff J, TANDEM c (2021). “Impact of Intermediate Hyperglycemia and Diabetes
on Immune Dysfunction in Tuberculosis.” _Clin Infect Dis_, *72*, 69-78. doi:10.1093/cid/ciaa751  [☞ Link](https://doi.org/10.1093/cid/ciaa751)

Wyler E, Mösbauer K, Franke V, Diag A, Gottula L, Arsiè R, Klironomos F, Koppstein D, Hönzke K, Ayoub S, Buccitelli C, Hoffmann K, Richter A, Legnini I,
**Ivanov  A**, Mari T, Del GS, Papies J, Praktiknjo S, Meyer T, Müller M, Niemeyer D, Hocke A, Selbach M, Akalin A, Rajewsky N, Drosten C, Landthaler M
(2021). “Transcriptomic profiling of SARS-CoV-2 infected human cell lines identifies HSP90 as target for COVID-19 therapy.” _iScience_, *24*, 102151.
doi:10.1016/j.isci.2021.102151  [☞ Link](https://doi.org/10.1016/j.isci.2021.102151)

Mölder F, Jablonski K, Letcher B, Hall M, Tomkins-Tinch C, Sochat V, Forster J, Lee S, Twardziok S, Kanitz A, Wilm A, **Holtgrewe  M**, Rahmann S, Nahnsen S,
Köster J (2021). “Sustainable data analysis with Snakemake.” _F1000Res_, *10*, 33. doi:10.12688/f1000research.29032.2
 [☞ Link](https://doi.org/10.12688/f1000research.29032.2)

**Messerschmidt  C**, Foddis M, Blumenau S, Müller S, **Bentele  K**, **Holtgrewe  M**, Kun-Rodrigues C, Alonso I, do CMM, Morgadinho A, Velon A, Santo G, Santana I,
Mönkäre S, Kuuluvainen L, Schleutker J, Pöyhönen M, Myllykangas L, Senatore A, Berchtold D, Winek K, Meisel A, Pavlovic A, Kostic V, Dobricic V, Lohmann
E, Hanagasi H, Guven G, Bilgic B, Bras J, Guerreiro R, **Beule  D**, Dirnagl U, Sassi C (2021). “PHACTR1 genetic variability is not critical in small vessel
ischemic disease patients and PcomA recruitment in C57BL/6J mice.” _Sci Rep_, *11*(1), 6072. doi:10.1038/s41598-021-84919-x
 [☞ Link](https://doi.org/10.1038/s41598-021-84919-x)

Zarate Y, Uehara T, Abe K, Oginuma M, Harako S, Ishitani S, Lehesjoki A, Bierhals T, Kloth K, Ehmke N, Horn D, **Holtgrewe  M**, Anderson K, Viskochil D,
Edgar-Zarate C, Sacoto M, Schnur R, Morrow M, Sanchez-Valle A, Pappas J, Rabin R, Muona M, Anttonen A, Platzer K, Luppe J, Gburek-Augustat J, Kaname T,
Okamoto N, Mizuno S, Kaido Y, Ohkuma Y, Hirose Y, Ishitani T, Kosaki K (2021). “CDK19-related disorder results from both loss-of-function and
gain-of-function de novo missense variants.” _Genet Med_, *23*(6), 1050-1057. doi:10.1038/s41436-020-01091-9
 [☞ Link](https://doi.org/10.1038/s41436-020-01091-9)

Vogt G, El CN, Herczegfalvi Á, Kölbel H, Lekaj A, Schara U, **Holtgrewe  M**, Krause S, Horvath R, Schuelke M, Hübner C, Mundlos S, Roos A, Lochmüller H,
Karcagi V, Kornak U, Fischer-Zirnsak B (2021). “Expanding the clinical and molecular spectrum of ATP6V1A related metabolic cutis laxa.” _J Inherit Metab
Dis_, *44*(4), 972-986. doi:10.1002/jimd.12341  [☞ Link](https://doi.org/10.1002/jimd.12341)

Yildirim F, Foddis M, Blumenau S, Müller S, Kajetan B, **Holtgrewe  M**, Kola V, **Beule  D**, Sassi C (2021). “Shared and oppositely regulated transcriptomic
signatures in Huntington's disease and brain ischemia confirm known and unveil novel potential neuroprotective genes.” _Neurobiol Aging_, *104*,
122.e1-122.e17. doi:10.1016/j.neurobiolaging.2021.03.001  [☞ Link](https://doi.org/10.1016/j.neurobiolaging.2021.03.001)

Elsner J, Mensah M, **Holtgrewe  M**, Hertzberg J, Bigoni S, Busche A, Coutelier M, de SD, Elçioglu N, Filges I, Gerkes E, Girisha K, Graul-Neumann L,
Jamsheer A, Krawitz P, Kurth I, Markus S, Megarbane A, Reis A, Reuter M, Svoboda D, Teller C, Tuysuz B, Türkmen S, Wilson M, Woitschach R, Vater I,
Caliebe A, Hülsemann W, Horn D, Mundlos S, Spielmann M (2021). “Genome sequencing in families with congenital limb malformations.” _Hum Genet_,
*140*(8), 1229-1239. doi:10.1007/s00439-021-02295-y  [☞ Link](https://doi.org/10.1007/s00439-021-02295-y)

Seidel F, **Holtgrewe  M**, Al-Wakeel-Marquard N, Opgen-Rhein B, Dartsch J, Herbst C, **Beule  D**, Pickardt T, Klingel K, Messroghli D, Berger F, Schubert S,
Kühnisch J, Klaassen S (2021). “Pathogenic Variants Associated With Dilated Cardiomyopathy Predict Outcome in Pediatric Myocarditis.” _Circ Genom Precis
Med_, *14*(4), e003250. doi:10.1161/CIRCGEN.120.003250  [☞ Link](https://doi.org/10.1161/CIRCGEN.120.003250)

Melo U, Piard J, Fischer-Zirnsak B, Klever M, Schöpflin R, Mensah M, **Holtgrewe  M**, Arbez-Gindre F, Martin A, Guigue V, Gaillard D, Landais E, Roze V,
Kremer V, Ramanah R, Cabrol C, Harms F, Kornak U, Spielmann M, Mundlos S, van Maldergem L (2021). “Complete lung agenesis caused by complex genomic
rearrangements with neo-TAD formation at the SHH locus.” _Hum Genet_, *140*(10), 1459-1469. doi:10.1007/s00439-021-02344-6
 [☞ Link](https://doi.org/10.1007/s00439-021-02344-6)

Sczakiel H, Hülsemann W, **Holtgrewe  M**, Abad-Perez A, Elsner J, Schwartzmann S, Horn D, Spielmann M, Mundlos S, Mensah M (2021). “GLI3 variants causing
isolated polysyndactyly are not restricted to the protein's C-terminal third.” _Clin Genet_, *100*(6), 758-765. doi:10.1111/cge.14059
 [☞ Link](https://doi.org/10.1111/cge.14059)


### 2020


Mattei D, **Ivanov  A**, van Oostrum M, Pantelyushin S, Richetto J, Mueller F, Beffinger M, Schellhammer L, Vom BJ, Wollscheid B, **Beule  D**, Paolicelli R,
Meyer U (2020). “Enzymatic Dissociation Induces Transcriptional and Proteotype Bias in Brain Cell Populations.” _Int J Mol Sci_, *21*.
doi:10.3390/ijms21217944  [☞ Link](https://doi.org/10.3390/ijms21217944)

**Kuhring  M**, Eisenberger A, Schmidt V, Kränkel N, Leistner D, Kirwan J, **Beule  D** (2020). “Concepts and Software Package for Efficient Quality Control in
Targeted Metabolomics Studies: MeTaQuaC.” _Anal Chem_, *92*, 10241-10245. doi:10.1021/acs.analchem.0c00136
 [☞ Link](https://doi.org/10.1021/acs.analchem.0c00136)

Koche R, Rodriguez-Fos E, Helmsauer K, Burkert M, MacArthur I, Maag J, Chamorro R, Munoz-Perez N, Puiggròs M, Dorado GH, Bei Y, Röefzaad C, Bardinet V,
Szymansky A, Winkler A, Thole T, Timme N, Kasack K, Fuchs S, Klironomos F, Thiessen N, **Blanc  E**, Schmelz K, Künkele A, Hundsdörfer P, Rosswog C, Theissen
J, **Beule  D**, Deubzer H, Sauer S, Toedling J, Fischer M, Hertwig F, Schwarz R, Eggert A, Torrents D, Schulte J, Henssen A (2020). “Extrachromosomal
circular DNA drives oncogenic genome remodeling in neuroblastoma.” _Nat Genet_, *52*, 29-34. doi:10.1038/s41588-019-0547-z
 [☞ Link](https://doi.org/10.1038/s41588-019-0547-z)

**Holtgrewe  M**, **Stolpe  O**, **Nieminen  M**, Mundlos S, Knaus A, Kornak U, Seelow D, Segebrecht L, Spielmann M, Fischer-Zirnsak B, Boschann F, Scholl U, Ehmke N,
**Beule  D** (2020). “VarFish: comprehensive DNA variant analysis for diagnostics and research.” _Nucleic Acids Res_, *48*, W162-W169.
doi:10.1093/nar/gkaa241  [☞ Link](https://doi.org/10.1093/nar/gkaa241)

**Obermayer  B**, **Holtgrewe  M**, **Nieminen  M**, **Messerschmidt  C**, **Beule  D** (2020). “SCelVis: exploratory single cell data analysis on the desktop and in the cloud.”
_PeerJ_, *8*, e8607. doi:10.7717/peerj.8607  [☞ Link](https://doi.org/10.7717/peerj.8607)

**Nieminen  M**, **Stolpe  O**, Schumann F (2020). “SODAR Core: a Django-based framework for scientific data management and analysis web apps.” _Journal of Open
Source Software_, *5*(55), 1584. doi:10.21105/joss.01584  [☞ Link](http://edoc.mdc-berlin.de/19748/1/19748oa.pdf)

Akpa C, Kleo K, Oker E, Tomaszewski N, **Messerschmidt  C**, López C, Wagener R, Oehl-Huber K, Dettmer K, Schoeler A, Lenze D, Oefner P, **Beule  D**, Siebert R,
Capper D, Dimitrova L, Hummel M (2020). “Acquired resistance to DZNep-mediated apoptosis is associated with copy number gains of AHCY in a B-cell
lymphoma model.” _BMC Cancer_, *20*, 427. doi:10.1186/s12885-020-06937-8  [☞ Link](https://doi.org/10.1186/s12885-020-06937-8)

**Messerschmidt  C**, **Obermayer  B**, Klinghammer K, Ochsenreither S, Treue D, Stenzinger A, Glimm H, Fröhling S, Kindler T, Brandts C, Schulze-Osthoff K,
Weichert W, Tinhofer I, Klauschen F, Keilholz U, **Beule  D**, Rieke D (2020). “Distinct immune evasion in APOBEC-enriched, HPV-negative HNSCC.” _Int J
Cancer_, *147*, 2293-2302. doi:10.1002/ijc.33123  [☞ Link](https://doi.org/10.1002/ijc.33123)

Boschann F, Fischer-Zirnsak B, Wienker T, **Holtgrewe  M**, Seelow D, Eichhorn B, Döhnert S, Fahsold R, Horn D, Graul-Neumann L (2020). “An intronic splice
site alteration in combination with a large deletion affecting VPS13B (COH1) causes Cohen syndrome.” _Eur J Med Genet_, *63*, 103973.
doi:10.1016/j.ejmg.2020.103973  [☞ Link](https://doi.org/10.1016/j.ejmg.2020.103973)

Rieke D, Klinghammer K, **Obermayer  B**, **Blanc  E**, **Messerschmidt  C**, Jöhrens K, Klauschen F, Tinhofer I, **Beule  D**, Keilholz U, Ochsenreither S (2020).
“Association of a STK11/KEAP1-Mutation Gene Expression Signature in Lung Adenocarcinoma with Immune Desertion in Squamous Cell Carcinomas and Mediation
by NFE2L2 Deregulation.” _J Clin Oncol_, *38*(15_suppl), 3082-3082. doi:10.1200/JCO.2020.38.15_suppl.3082
 [☞ Link](https://doi.org/10.1200/JCO.2020.38.15_suppl.3082)

Ehmke N, Cusmano-Ozog K, Koenig R, **Holtgrewe  M**, Nur B, Mihci E, Babcock H, Gonzaga-Jauregui C, Overton J, Xiao J, Martinez A, Muenke M, Balzer A, Jochim
J, El CN, Fischer-Zirnsak B, Huber C, Kornak U, Elsea S, Cormier-Daire V, Ferreira C (2020). “Biallelic variants in KYNU cause a multisystemic syndrome
with hand hyperphalangism.” _Bone_, *133*, 115219. doi:10.1016/j.bone.2019.115219  [☞ Link](https://doi.org/10.1016/j.bone.2019.115219)

Seidel E, Walenda G, **Messerschmidt  C**, **Obermayer  B**, Peitzsch M, Wallace P, Bahethi R, Yoo T, Choi M, Schrade P, Bachmann S, Liebisch G, Eisenhofer G,
**Beule  D**, Scholl U (2020). “Generation and characterization of a mitotane-resistant adrenocortical cell line.” _Endocr Connect_, *9*, 122-134.
doi:10.1530/EC-19-0510  [☞ Link](https://doi.org/10.1530/EC-19-0510)

Melo U, Schöpflin R, Acuna-Hidalgo R, Mensah M, Fischer-Zirnsak B, **Holtgrewe  M**, Klever M, Türkmen S, Heinrich V, Pluym I, Matoso E, Bernardo dSS, Louro
P, Hülsemann W, Cohen M, Dufke A, Latos-Bieleńska A, Vingron M, Kalscheuer V, Quintero-Rivera F, Spielmann M, Mundlos S (2020). “Hi-C Identifies Complex
Genomic Rearrangements and TAD-Shuffling in Developmental Diseases.” _Am J Hum Genet_, *106*, 872-884. doi:10.1016/j.ajhg.2020.04.016
 [☞ Link](https://doi.org/10.1016/j.ajhg.2020.04.016)

Robinson P, Ravanmehr V, Jacobsen J, Danis D, Zhang X, Carmody L, Gargano M, Thaxton C, UNC BC, Karlebach G, Reese J, **Holtgrewe  M**, Köhler S, McMurry J,
Haendel M, Smedley D (2020). “Interpretable Clinical Genomics with a Likelihood Ratio Paradigm.” _Am J Hum Genet_, *107*, 403-417.
doi:10.1016/j.ajhg.2020.06.021  [☞ Link](https://doi.org/10.1016/j.ajhg.2020.06.021)

Blumenau S, Foddis M, Müller S, **Holtgrewe  M**, **Bentele  K**, Berchtold D, **Beule  D**, Dirnagl U, Sassi C (2020). “Investigating APOE, APP-Aβ metabolism genes
and Alzheimer's disease GWAS hits in brain small vessel ischemic disease.” _Sci Rep_, *10*, 7103. doi:10.1038/s41598-020-63183-5
 [☞ Link](https://doi.org/10.1038/s41598-020-63183-5)

Quade A, Thiel A, Kurth I, **Holtgrewe  M**, Elbracht M, **Beule  D**, Eggermann K, Scholl U, Häusler M (2020). “Paroxysmal tonic upgaze: A heterogeneous clinical
condition responsive to carbonic anhydrase inhibition.” _Eur J Paediatr Neurol_, *25*, 181-186. doi:10.1016/j.ejpn.2019.11.002
 [☞ Link](https://doi.org/10.1016/j.ejpn.2019.11.002)

Thole T, Toedling J, Sprüssel A, Pfeil S, Savelyeva L, Capper D, **Messerschmidt  C**, **Beule  D**, Groeneveld-Krentz S, Eckert C, Gambara G, Henssen A, Finkler
S, Schulte J, Sieber A, Bluethgen N, Regenbrecht C, Künkele A, Lodrini M, Eggert A, Deubzer H (2020). “Reflection of neuroblastoma intratumor
heterogeneity in the new OHC-NB1 disease model.” _Int J Cancer_, *146*, 1031-1041. doi:10.1002/ijc.32572  [☞ Link](https://doi.org/10.1002/ijc.32572)

Lamping M, **Benary  M**, Leyvraz S, **Messerschmidt  C**, **Blanc  E**, Kessler T, Schütte M, Lenze D, Jöhrens K, Burock S, Klinghammer K, Ochsenreither S, Sers C,
Schäfer R, Tinhofer I, **Beule  D**, Klauschen F, Yaspo M, Keilholz U, Rieke D (2020). “Support of a molecular tumour board by an evidence-based decision
management system for precision oncology.” _Eur J Cancer_, *127*, 41-51. doi:10.1016/j.ejca.2019.12.017  [☞ Link](https://doi.org/10.1016/j.ejca.2019.12.017)

Praktiknjo S, **Obermayer  B**, Zhu Q, Fang L, Liu H, Quinn H, Stoeckius M, Kocks C, Birchmeier W, Rajewsky N (2020). “Tracing tumorigenesis in a solid tumor
model at single-cell resolution.” _Nat Commun_, *11*, 991. doi:10.1038/s41467-020-14777-0  [☞ Link](https://doi.org/10.1038/s41467-020-14777-0)

Suter A, Santos-Simarro F, Toerring P, Abad PA, Ramos-Mejia R, Heath K, Huckstadt V, Parrón-Pajares M, Mensah M, Hülsemann W, **Holtgrewe  M**, Mundlos S,
Kornak U, Bartsch O, Ehmke N (2020). “Variable pulmonary manifestations in Chitayat syndrome: Six additional affected individuals.” _Am J Med Genet A_,
*182*, 2068-2076. doi:10.1002/ajmg.a.61735  [☞ Link](https://doi.org/10.1002/ajmg.a.61735)

Bockstahler M, Fischer A, Goetzke C, Neumaier H, Sauter M, Kespohl M, Müller A, Meckes C, Salbach C, Schenk M, Heuser A, Landmesser U, **Weiner  J**, Meder
B, Lehmann L, Kratzer A, Klingel K, Katus H, Kaya Z, Beling A (2020). “Heart-Specific Immune Responses in an Animal Model of Autoimmune-Related
Myocarditis Mitigated by an Immunoproteasome Inhibitor and Genetic Ablation.” _Circulation_, *141*, 1885-1902. doi:10.1161/CIRCULATIONAHA.119.043171
 [☞ Link](https://doi.org/10.1161/CIRCULATIONAHA.119.043171)

Krishnamoorthy G, Kaiser P, Abu AU, **Weiner  J3**, Moura-Alves P, Brinkmann V, Kaufmann S (2020). “FX11 limits Mycobacterium tuberculosis growth and
potentiates bactericidal activity of isoniazid through host-directed activity.” _Dis Model Mech_, *13*. doi:10.1242/dmm.041954
 [☞ Link](https://doi.org/10.1242/dmm.041954)

Tregoning J, **Weiner  J**, Cizmeci D, Hake D, Maertzdorf J, Kaufmann S, Leroux-Roels G, Maes C, Aerssens A, Calvert A, Jones C (2020). “Pregnancy has a
minimal impact on the acute transcriptional signature to vaccination.” _NPJ Vaccines_, *5*, 29. doi:10.1038/s41541-020-0177-6
 [☞ Link](https://doi.org/10.1038/s41541-020-0177-6)

**Weiner  J**, Domaszewska T, Donkor S, Kaufmann S, Hill P, Sutherland J (2020). “Changes in Transcript, Metabolite, and Antibody Reactivity During the Early
Protective Immune Response in Humans to Mycobacterium tuberculosis Infection.” _Clin Infect Dis_, *71*, 30-40. doi:10.1093/cid/ciz785
 [☞ Link](https://doi.org/10.1093/cid/ciz785)

Win Z, **Weiner  J**, Listanco A, Patel N, Sharma R, Greenwood A, Maertzdorf J, Mollenkopf H, Pizzoferro K, Cole T, Bodinham C, Kaufmann S, Denoel P, Del GG,
Lewis D (2020). “Systematic Evaluation of Kinetics and Distribution of Muscle and Lymph Node Activation Measured by (18)F-FDG- and (11)C-PBR28-PET/CT
Imaging, and Whole Blood and Muscle Transcriptomics After Immunization of Healthy Humans With Adjuvanted and Unadjuvanted Vaccines.” _Front Immunol_,
*11*, 613496. doi:10.3389/fimmu.2020.613496  [☞ Link](https://doi.org/10.3389/fimmu.2020.613496)

Mochel F, Rastetter A, Ceulemans B, Platzer K, Yang S, Shinde D, Helbig K, Lopergolo D, Mari F, Renieri A, Benetti E, Canitano R, Waisfisz Q, Plomp A,
Huisman S, Wilson G, Cathey S, Louie R, Gaudio D, Waggoner D, Kacker S, Nugent K, Roeder E, Bruel A, Thevenon J, Ehmke N, Horn D, **Holtgrewe  M**, Kaiser F,
Kamphausen S, Abou JR, Weckhuysen S, Dalle C, Depienne C (2020). “Variants in the SK2 channel gene (KCNN2) lead to dominant neurodevelopmental movement
disorders.” _Brain_, *143*(12), 3564-3573. doi:10.1093/brain/awaa346  [☞ Link](https://doi.org/10.1093/brain/awaa346)

Foddis M, Winek K, **Bentele  K**, Mueller S, Blumenau S, Reichhart NN, Crespo-Garcia S, Harnett D, **Ivanov  A**, Meisel A, Joussen A, Strauss O, **Beule  D**,
Dirnagl U, Sassi C (2020). “An exploratory investigation of brain collateral circulation plasticity after cerebral ischemia in two experimental C57BL/6
mouse models.” _J Cereb Blood Flow Metab_, *40*, 276-287. doi:10.1177/0271678X19827251  [☞ Link](https://doi.org/10.1177/0271678X19827251)

Quade A, Thiel A, Kurth I, **Holtgrewe  M**, Elbracht M, **Beule  D**, Eggermann K, Scholl U, Häusler M (2020). “Paroxysmal tonic upgaze: A heterogeneous clinical
condition responsive to carbonic anhydrase inhibition.” _Eur J Paediatr Neurol_, *25*, 181-186. doi:10.1016/j.ejpn.2019.11.002
 [☞ Link](https://doi.org/10.1016/j.ejpn.2019.11.002)


### 2019


**Holtgrewe  M**, **Messerschmidt  C**, **Nieminen  M**, **Beule  D** (2019). “Digestiflow: From BCL to FASTQ with Ease.” _Bioinformatics_.

Schumann F, **Blanc  E**, **Messerschmidt  C**, Blankenstein T, Busse A, **Beule  D** (2019). “SigsPack, a package for cancer mutational signatures.” _BMC
Bioinformatics_, *20*, 450. doi:10.1186/s12859-019-3043-7  [☞ Link](https://doi.org/10.1186/s12859-019-3043-7)

**Blanc  E**, **Holtgrewe  M**, Dhamodaran A, **Messerschmidt  C**, Willimsky G, Blankenstein T, **Beule  D** (2019). “Identification and ranking of recurrent neo-epitopes
in cancer.” _BMC Med Genomics_, *12*, 171. doi:10.1186/s12920-019-0611-7  [☞ Link](https://doi.org/10.1186/s12920-019-0611-7)

Boeddrich A, Babila J, Wiglenda T, Diez L, Jacob M, Nietfeld W, Huska M, Haenig C, Groenke N, Buntru A, **Blanc  E**, Meier J, Vannoni E, Erck C, Friedrich
B, Martens H, Neuendorf N, Schnoegl S, Wolfer D, Loos M, **Beule  D**, Andrade-Navarro M, Wanker E (2019). “The Anti-amyloid Compound DO1 Decreases Plaque
Pathology and Neuroinflammation-Related Expression Changes in 5xFAD Transgenic Mice.” _Cell Chem Biol_, *26*, 109-120.e7.
doi:10.1016/j.chembiol.2018.10.013  [☞ Link](https://doi.org/10.1016/j.chembiol.2018.10.013)

Borgsmüller N, Gloaguen Y, Opialla T, **Blanc  E**, Sicard E, Royer A, Le BB, Durand S, Migné C, Pétéra M, Pujos-Guillot E, Giacomoni F, Guitton Y, **Beule  D**,
Kirwan J (2019). “WiPP: Workflow for Improved Peak Picking for Gas Chromatography-Mass Spectrometry (GC-MS) Data.” _Metabolites_, *9*.
doi:10.3390/metabo9090171  [☞ Link](https://doi.org/10.3390/metabo9090171)

Buonfiglioli A, Efe I, Guneykaya D, **Ivanov  A**, Huang Y, Orlowski E, Krüger C, Deisz R, Markovic D, Flüh C, Newman A, Schneider U, **Beule  D**, Wolf S, Dzaye
O, Gutmann D, Semtner M, Kettenmann H, Lehnardt S (2019). “let-7 MicroRNAs Regulate Microglial Function and Suppress Glioma Growth through Toll-Like
Receptor 7.” _Cell Rep_, *29*, 3460-3471.e7. doi:10.1016/j.celrep.2019.11.029  [☞ Link](https://doi.org/10.1016/j.celrep.2019.11.029)

Kühnisch J, Herbst C, Al-Wakeel-Marquard N, Dartsch J, **Holtgrewe  M**, Baban A, Mearini G, Hardt J, Kolokotronis K, Gerull B, Carrier L, **Beule  D**, Schubert
S, Messroghli D, Degener F, Berger F, Klaassen S (2019). “Targeted panel sequencing in pediatric primary cardiomyopathy supports a critical role of
TNNI3.” _Clin Genet_, *96*, 549-559. doi:10.1111/cge.13645  [☞ Link](https://doi.org/10.1111/cge.13645)

**Kuhring  M**, **Nieminen  M**, Kirwan J, **Beule  D**, **Holtgrewe  M** (2019). “AltamISA: A Python API for ISA-Tab Files.” _JOSS_, *4*, 1610. doi:10.21105/joss.01610


Lamping M, Rieke D, Klauschen F, Jöhrens K, Anagnostopoulos I, Lenze D, Tinhofer I, **Benary  M**, Ochsenreither S, Klinghammer K, Burock S, Jann H, Stüven
A, Ditzen D, **Beule  D**, **Messerschmidt  C**, **Blanc  E**, Schäfer R, Keilholz U (2019). “Clinical Impact of Comprehensive versus Targeted Genomic Analysis for
Precision Oncology.” _J Clin Oncol_, *37*(15_suppl), e13033-e13033. doi:10.1200/JCO.2019.37.15_suppl.e13033
 [☞ Link](https://doi.org/10.1200/JCO.2019.37.15_suppl.e13033)

Herda S, Heimann A, Althoff S, Ruß J, Bullinger L, **Beule  D**, **Obermayer  B**, Na I (2019). “Long-Term T Cell Expansion Results in Increased Numbers of
Central Memory T Cells with Sustained Functional Properties for Adoptive T Cell Therapy.” _Blood_. doi:10.1182/blood-2019-124736
 [☞ Link](https://www.sciencedirect.com/science/article/pii/S0006497118598608)

van~Heesch S, Witte F, Schneider-Lunitz V, Schulz J, Adami E, Faber A, Kirchner M, Maatz H, Blachut S, Sandmann C, Kanda M, Worth C, Schafer S,
Calviello L, Merriott R, Patone G, Hummel O, Wyler E, **Obermayer  B**, Mücke M, Lindberg E, Trnka F, Memczak S, Schilling M, Felkin L, Barton P, Quaife N,
Vanezis K, Diecke S, Mukai M, Mah N, Oh S, Kurtz A, Schramm C, Schwinge D, Sebode M, Harakalova M, Asselbergs F, Vink A, de WR, Viswanathan S, Widjaja
A, Gärtner-Rommel A, Milting H, Dos RC, Knosalla C, Mertins P, Landthaler M, Vingron M, Linke W, Seidman J, Seidman C, Rajewsky N, Ohler U, Cook S,
Hubner N (2019). “The Translational Landscape of the Human Heart.” _Cell_, *178*, 242-260.e29. doi:10.1016/j.cell.2019.05.010
 [☞ Link](https://doi.org/10.1016/j.cell.2019.05.010)


### 2018


**Holtgrewe  M**, Knaus A, Hildebrand G, Pantel J, Santos M, Neveling K, Goldmann J, Schubach M, Jäger M, Coutelier M, Mundlos S, **Beule  D**, Sperling K,
Krawitz P (2018). “Multisite de novo mutations in human offspring after paternal exposure to ionizing radiation.” _Sci Rep_, *8*, 14611.
doi:10.1038/s41598-018-33066-x  [☞ Link](https://doi.org/10.1038/s41598-018-33066-x)

Guneykaya D, **Ivanov  A**, Hernandez D, Haage V, Wojtas B, Meyer N, Maricos M, Jordan P, Buonfiglioli A, Gielniewski B, Ochocka N, Cömert C, Friedrich C,
Artiles L, Kaminska B, Mertins P, **Beule  D**, Kettenmann H, Wolf S (2018). “Transcriptional and Translational Differences of Microglia from Male and Female
Brains.” _Cell Rep_, *24*, 2773-2783.e6. doi:10.1016/j.celrep.2018.08.001  [☞ Link](https://doi.org/10.1016/j.celrep.2018.08.001)

Rieke D, Lamping M, Klauschen F, Ochsenreither S, Schütte M, Kessler T, Klinghammer K, Jöhrens K, **Messerschmidt  C**, Lenze D, Burock S, Ditzen D, Schäfer
R, Pavel M, Tinhofer I, Sers C, **Beule  D**, Yaspo M, Leyvraz S, Keilholz U (2018). “Efficacy of a Structured Workflow for the Interpretation of
Comprehensive Genomic Analysis Data in Clinical Routine.” _J Clin Oncol_, *36*(15_suppl), e24164-e24164. doi:10.1200/JCO.2018.36.15_suppl.e24164
 [☞ Link]( https://doi.org/10.1200/JCO.2018.36.15_suppl.e24164 )

Salatzki J, Foryst-Ludwig A, **Bentele  K**, Blumrich A, Smeir E, Ban Z, Brix S, Grune J, Beyhoff N, Klopfleisch R, Dunst S, Surma M, Klose C, Rothe M,
Heinzel F, Krannich A, Kershaw E, **Beule  D**, Schulze P, Marx N, Kintscher U (2018). “Adipose tissue ATGL modifies the cardiac lipidome in
pressure-overload-induced left ventricular failure.” _PLoS Genet_, *14*, e1007171. doi:10.1371/journal.pgen.1007171
 [☞ Link](https://doi.org/10.1371/journal.pgen.1007171)


### 2017


Kammertoens T, Friese C, Arina A, Idel C, Briesemeister D, Rothe M, **Ivanov  A**, Szymborska A, Patone G, Kunz S, Sommermeyer D, Engels B, Leisegang M,
Textor A, Fehling H, Fruttiger M, Lohoff M, Herrmann A, Yu H, Weichselbaum R, Uckert W, Hübner N, Gerhardt H, **Beule  D**, Schreiber H, Blankenstein T
(2017). “Tumour ischaemia by interferon-γ resembles physiological blood vessel regression.” _Nature_, *545*, 98-102. doi:10.1038/nature22311
 [☞ Link](https://doi.org/10.1038/nature22311)

Mamlouk S, Childs L, Aust D, Heim D, Melching F, Oliveira C, Wolf T, Durek P, Schumacher D, Bläker H, von WM, Gastl B, Möhr K, Menne A, Zeugner S,
Redmer T, Lenze D, Tierling S, Möbs M, Weichert W, Folprecht G, **Blanc  E**, **Beule  D**, Schäfer R, Morkel M, Klauschen F, Leser U, Sers C (2017). “DNA copy
number changes define spatial patterns of heterogeneity in colorectal cancer.” _Nat Commun_, *8*, 14093. doi:10.1038/ncomms14093
 [☞ Link](https://doi.org/10.1038/ncomms14093)

Mattei D, **Ivanov  A**, Ferrai C, Jordan P, Guneykaya D, Buonfiglioli A, Schaafsma W, Przanowski P, Deuther-Conrad W, Brust P, Hesse S, Patt M, Sabri O,
Ross T, Eggen B, Boddeke E, Kaminska B, **Beule  D**, Pombo A, Kettenmann H, Wolf S (2017). “Maternal immune activation results in complex microglial
transcriptome signature in the adult offspring that is reversed by minocycline treatment.” _Transl Psychiatry_, *7*, e1120. doi:10.1038/tp.2017.80
 [☞ Link](https://doi.org/10.1038/tp.2017.80)

**Messerschmidt  C**, **Holtgrewe  M**, **Beule  D** (2017). “HLA-MA: simple yet powerful matching of samples using HLA typing results.” _Bioinformatics_, *33*,
2241-2242. doi:10.1093/bioinformatics/btx132  [☞ Link](https://doi.org/10.1093/bioinformatics/btx132)

Rieke D, **Messerschmidt  C**, Ochsenreither S, Klinghammer K, Tinhofer I, Jöhrens K, Klauschen F, **Beule  D**, Keilholz U (2017). “Association of an APOBEC
Mutational Signature, Mutational Load, and BRCAness with Inflammation and PD-L1 Expression in HNSCC.” _J Clin Oncol_, *35*(15_suppl), e14613-e14613.
doi:10.1200/JCO.2017.35.15_suppl.e14613  [☞ Link](https://doi.org/10.1200/JCO.2017.35.15_suppl.e14613)

Yaakov G, Lerner D, **Bentele  K**, Steinberger J, Barkai N (2017). “Coupling phenotypic persistence to DNA damage increases genetic diversity in severe
stress.” _Nat Ecol Evol_, *1*, 16. doi:10.1038/s41559-016-0016  [☞ Link](https://doi.org/10.1038/s41559-016-0016)


### 2016


**Holtgrewe  M**, **Beule  D** (2016). “VCFPy: A Python 3 Library with Good Support for Both Reading and Writing VCF.” _JOSS_, *1*(6), 85. doi:10.21105/joss.00085
 [☞ Link](https://doi.org/10.21105/joss.00085)
