Baumgardt M, Hülsemann M, Löwa A, Fatykhova D, Hoffmann K, Kessler M, Mieth M, Hellwig K, Frey D, Langenhagen A, Voss A, **Obermayer B**, Wyler E, Dökel S,
Gruber AD, Tölch U, Hippenstiel S, Hocke AC, Hönzke K (2022). “State-of-the-art analytical methods of viral infections in human lung organoids.” _PLOS
ONE_, *17*(12), 1-8. doi:10.1371/journal.pone.0276115  [☞ Link](https://doi.org/10.1371/journal.pone.0276115)

Pennitz P, Kirsten H, Friedrich VD, Wyler E, Goekeri C, **Obermayer B**, Heinz GA, Mashreghi M, B"uttner M, Trimpert J, Landthaler M, Suttorp N, Hocke AC,
Hippenstiel S, T"onnies M, Scholz M, Kuebler WM, Witzenrath M, Hoenzke K, Nouailles G (2022). “A pulmonologist's guide to perform and analyse
cross-species single lung cell transcriptomics.” _European Respiratory Review_, *31*(165). ISSN 0905-9180, doi:10.1183/16000617.0056-2022
 [☞ Link](https://err.ersjournals.com/content/31/165/220056)

Bertrams W, Hönzke K, **Obermayer B**, Tönnies M, Bauer TT, Schneider P, Neudecker J, Rückert JC, Stiewe T, Nist A, Eggeling S, Suttorp N, Wolff T,
Hippenstiel S, Schmeck B, Hocke AC (2022). “Transcriptomic comparison of primary human lung cells with lung tissue samples and the human A549 lung cell
line highlights cell type specific responses during infections with influenza A virus.” _Scientific Reports_, *12*(1), 20608. ISSN 2045-2322,
doi:10.1038/s41598-022-24792-4  [☞ Link](https://doi.org/10.1038/s41598-022-24792-4)

Jarick K, Topczewska P, Jakob M, Yano H, Arifuzzaman M, Gao X, Boulekou S, Stokic-Trtica V, Leclère P, Preußer A, Rompe Z, Stamm A, Tsou A, Chu C,
Heinrich F, Guerra G, Durek P, **Ivanov A**, **Beule D**, Helfrich S, Duerr C, Kühl A, Stehle C, Romagnani C, Mashreghi M, Diefenbach A, Artis D, Klose C
(2022). “Non-redundant functions of group 2 innate lymphoid cells.” _Nature_, *611*(7937), 794-800. doi:10.1038/s41586-022-05395-5
 [☞ Link](https://doi.org/10.1038/s41586-022-05395-5)

**Ivanov A**, Mattei D, Radscheit K, Compagnion A, **Pett J**, Herzel H, Paolicelli R, Piwecka M, Meyer U, **Beule D** (2022). “Analyses of circRNA Expression
throughout the Light-Dark Cycle Reveal a Strong Regulation of Cdr1as, Associated with Light Entrainment in the SCN.” _Int J Mol Sci_, *23*.
doi:10.3390/ijms232012347  [☞ Link](https://doi.org/10.3390/ijms232012347)

Wirges A, Bunse M, Joedicke J, **Blanc E**, Gudipati V, Moles M, Shiku H, **Beule D**, Huppa J, Höpken U, Rehm A (2022). “EBAG9 silencing exerts an immune
checkpoint function without aggravating adverse effects.” _Mol Ther_, *30*, 3358-3378. doi:10.1016/j.ymthe.2022.07.009
 [☞ Link](https://doi.org/10.1016/j.ymthe.2022.07.009)

Hönzke K, **Obermayer B**, Mache C, Fathykova D, Kessler M, Dökel S, Wyler E, Baumgardt M, Löwa A, Hoffmann K, Graff P, Schulze J, Mieth M, Hellwig K, Demir
Z, Biere B, Brunotte L, Mecate-Zambrano A, Bushe J, Dohmen M, Hinze C, Elezkurtaj S, Tönnies M, Bauer T, Eggeling S, Tran H, Schneider P, Neudecker J,
Rückert J, Schmidt-Ott K, Busch J, Klauschen F, Horst D, Radbruch H, Radke J, Heppner F, Corman V, Niemeyer D, Müller M, Goffinet C, Mothes R,
Pascual-Reguant A, Hauser A, **Beule D**, Landthaler M, Ludwig S, Suttorp N, Witzenrath M, Gruber A, Drosten C, Sander L, Wolff T, Hippenstiel S, Hocke A
(2022). “Human lungs show limited permissiveness for SARS-CoV-2 due to scarce ACE2 levels but virus-induced expansion of inflammatory macrophages.” _Eur
Respir J_, *60*. doi:10.1183/13993003.02725-2021  [☞ Link](https://doi.org/10.1183/13993003.02725-2021)

Freitag K, Sterczyk N, Wendlinger S, **Obermayer B**, Schulz J, Farztdinov V, Mülleder M, Ralser M, Houtman J, Fleck L, Braeuning C, Sansevrino R, Hoffmann
C, Milovanovic D, Sigrist S, Conrad T, **Beule D**, Heppner F, Jendrach M (2022). “Spermidine reduces neuroinflammation and soluble amyloid beta in an
Alzheimer's disease mouse model.” _J Neuroinflammation_, *19*, 172. doi:10.1186/s12974-022-02534-7  [☞ Link](https://doi.org/10.1186/s12974-022-02534-7)

Schalbetter S, von Arx A, Cruz-Ochoa N, Dawson K, **Ivanov A**, Mueller F, Lin H, Amport R, Mildenberger W, Mattei D, **Beule D**, Földy C, Greter M, Notter T,
Meyer U (2022). “Adolescence is a sensitive period for prefrontal microglia to act on cognitive development.” _Sci Adv_, *8*, eabi6672.
doi:10.1126/sciadv.abi6672  [☞ Link](https://doi.org/10.1126/sciadv.abi6672)

Grunert C, Willimsky G, Peuker C, Rhein S, Hansmann L, Blankenstein T, **Blanc E**, **Beule D**, Keller U, Pezzutto A, Busse A (2022). “Isolation of
Neoantigen-Specific Human T Cell Receptors from Different Human and Murine Repertoires.” _Cancers (Basel)_, *14*. doi:10.3390/cancers14071842
 [☞ Link](https://doi.org/10.3390/cancers14071842)

Dannebaum R, Suwalski P, Asgharian H, Du ZG, Lin H, **Weiner J**, **Holtgrewe M**, Thibeault C, Müller M, Wang X, Karadeniz Z, Saccomanno J, Doehn J, Hübner R,
Hinzmann B, Blüher A, Siemann S, Telman D, Suttorp N, Witzenrath M, Hippenstiel S, Skurk C, Poller W, Sander L, **Beule D**, Kurth F, Guettouche T,
Landmesser U, Berka J, Luong K, Pa-COVID SG, Rubelt F, Heidecker B (2022). “Highly multiplexed immune repertoire sequencing links multiple lymphocyte
classes with severity of response to COVID-19.” _EClinicalMedicine_, *48*, 101438. doi:10.1016/j.eclinm.2022.101438
 [☞ Link](https://doi.org/10.1016/j.eclinm.2022.101438)

Horak P, Griffith M, Danos A, Pitel B, Madhavan S, Liu X, Chow C, Williams H, Carmody L, Barrow-Laing L, Rieke D, Kreutzfeldt S, Stenzinger A, Tamborero
D, **Benary M**, Rajagopal P, Ida C, Lesmana H, Satgunaseelan L, Merker J, Tolstorukov M, Campregher P, Warner J, Rao S, Natesan M, Shen H, Venstrom J, Roy
S, Tao K, Kanagal-Shamanna R, Xu X, Ritter D, Pagel K, Krysiak K, Dubuc A, Akkari Y, Li X, Lee J, King I, Raca G, Wagner A, Li M, Plon S, Kulkarni S,
Griffith O, Chakravarty D, Sonkin D (2022). “Standards for the classification of pathogenicity of somatic variants in cancer (oncogenicity): Joint
recommendations of Clinical Genome Resource (ClinGen), Cancer Genomics Consortium (CGC), and Variant Interpretation for Cancer Consortium (VICC).”
_Genet Med_, *24*, 986-998. doi:10.1016/j.gim.2022.01.001  [☞ Link](https://doi.org/10.1016/j.gim.2022.01.001)

**Weiner J**, **Obermayer B**, **Beule D** (2022). “Venn Diagrams May Indicate Erroneous Statistical Reasoning in Transcriptomics.” _Front Genet_, *13*, 818683.
doi:10.3389/fgene.2022.818683  [☞ Link](https://doi.org/10.3389/fgene.2022.818683)

Gloaguen Y, Kirwan J, **Beule D** (2022). “Deep Learning-Assisted Peak Curation for Large-Scale LC-MS Metabolomics.” _Anal Chem_, *94*, 4930-4937.
doi:10.1021/acs.analchem.1c02220  [☞ Link](https://doi.org/10.1021/acs.analchem.1c02220)

Chopra A, Mueller R, **Weiner J3**, Rosowski J, Dommisch H, Grohmann E, Schaefer A (2022). “BACH1 Binding Links the Genetic Risk for Severe Periodontitis
with ST8SIA1.” _J Dent Res_, *101*, 93-101. doi:10.1177/00220345211017510  [☞ Link](https://doi.org/10.1177/00220345211017510)

Bischoff P, Trinks A, Wiederspahn J, **Obermayer B**, **Pett J**, Jurmeister P, Elsner A, Dziodzio T, Rückert J, Neudecker J, Falk C, **Beule D**, Sers C, Morkel M,
Horst D, Klauschen F, Blüthgen N (2022). “The single-cell transcriptional landscape of lung carcinoid tumors.” _Int J Cancer_, *150*, 2058-2071.
doi:10.1002/ijc.33995  [☞ Link](https://doi.org/10.1002/ijc.33995)

Georg P, Astaburuaga-García R, Bonaguro L, Brumhard S, Michalick L, Lippert L, Kostevc T, Gäbel C, Schneider M, Streitz M, Demichev V, Gemünd I, Barone
M, Tober-Lau P, Helbig E, Hillus D, Petrov L, Stein J, Dey H, Paclik D, Iwert C, Mülleder M, Aulakh S, Djudjaj S, Bülow R, Mei H, Schulz A, Thiel A,
Hippenstiel S, Saliba A, Eils R, Lehmann I, Mall M, Stricker S, Röhmel J, Corman V, **Beule D**, Wyler E, Landthaler M, **Obermayer B**, von Stillfried S, Boor
P, Demir M, Wesselmann H, Suttorp N, Uhrig A, Müller-Redetzky H, Nattermann J, Kuebler W, Meisel C, Ralser M, Schultze J, Aschenbrenner A, Thibeault C,
Kurth F, Sander L, Blüthgen N, Sawitzki B, PA-COVID-19 SG (2022). “Complement activation induces excessive T cell cytotoxicity in severe COVID-19.”
_Cell_, *185*(3), 493-512.e25. ISSN 0092-8674, doi:10.1016/j.cell.2021.12.040  [☞ Link](https://doi.org/10.1016/j.cell.2021.12.040)

Zinnall U, **Milek M**, Minia I, Vieira-Vieira C, Müller S, Mastrobuoni G, Hazapis O, Del GS, Schwefel D, Bley N, Voigt F, Chao J, Kempa S, Hüttelmaier S,
Selbach M, Landthaler M (2022). “HDLBP binds ER-targeted mRNAs by multivalent interactions to promote protein synthesis of transmembrane and secreted
proteins.” _Nat Commun_, *13*(1), 2727. doi:10.1038/s41467-022-30322-7  [☞ Link](https://doi.org/10.1038/s41467-022-30322-7)

Dörr D, **Obermayer B**, **Weiner J**, Zimmermann K, Anania C, Wagner L, Lyras E, Sapozhnikova V, Lara-Astiaso D, Prósper F, Lang R, Lupiáñez D, **Beule D**, Höpken
U, Leutz A, Mildner A (2022). “C/EBPβ regulates lipid metabolism and Pparg isoform 2 expression in alveolar macrophages.” _Sci Immunol_, *7*(75),
eabj0140. doi:10.1126/sciimmunol.abj0140  [☞ Link](https://doi.org/10.1126/sciimmunol.abj0140)

Kedziora S, **Obermayer B**, Sugulle M, Herse F, Kräker K, Haase N, Langmia I, Müller D, Staff A, **Beule D**, Dechend R (2022). “Placental Transcriptome
Profiling in Subtypes of Diabetic Pregnancies Is Strongly Confounded by Fetal Sex.” _Int J Mol Sci_, *23*(23). ISSN 1422-0067, doi:10.3390/ijms232315388
 [☞ Link](https://doi.org/10.3390/ijms232315388)

Pennitz P, Kirsten H, Wyler E, Goekeri C, **Obermayer B**, Heinz GA, Mashreghi MF, Trimpert J, Landthaler M, Suttorp N, Hocke AC, Hippenstiel S, Tönnies M,
Scholz M, Kuebler WM, Witzenrath M, Hoenzke K, Nouailles G (2022). “Integrational approaches for cross-species analysis of lung pathologies at
single-cell resolution.” _Pneumologie_, *76*(S 01), FV 285. FV 285.

Hoffmann K, **Obermayer B**, Hönzke K, Fatykhova D, Demir Z, Löwa A, Alves L, Wyler E, Lopez-Rodriguez E, Mieth M, Baumgardt M, Hoppe J, Firsching T,
Tönnies M, Bauer T, Eggeling S, Tran H, Schneider P, Neudecker J, Rückert J, Gruber A, Ochs M, Landthaler M, **Beule D**, Suttorp N, Hippenstiel S, Hocke A,
Kessler M (2022). “Human alveolar progenitors generate dual lineage bronchioalveolar organoids.” _Commun Biol_, *5*(1), 875. ISSN 2399-3642,
doi:10.1038/s42003-022-03828-5  [☞ Link](https://doi.org/10.1038/s42003-022-03828-5)

Neitzel H, Varon R, Chughtai S, Dartsch J, Dutrannoy-Tönsing V, Nürnberg P, Nürnberg G, Schweiger M, Digweed M, Hildebrand G, Hackmann K, **Holtgrewe M**,
Sarioglu N, Schulze B, Horn D, Sperling K (2022). “Transmission ratio distortion of mutations in the master regulator of centriole biogenesis PLK4.”
_Hum Genet_, *141*(11), 1785-1794. doi:10.1007/s00439-022-02461-w  [☞ Link](https://doi.org/10.1007/s00439-022-02461-w)

Foddis M, Blumenau S, **Holtgrewe M**, Paquette K, Westra K, Alonso I, Macario M, Morgadinho A, Velon A, Santo G, Santana I, Mönkäre S, Kuuluvainen L,
Schleutker J, Pöyhönen M, Myllykangas L, Pavlovic A, Kostic V, Dobricic V, Lohmann E, Hanagasi H, Santos M, Guven G, Bilgic B, Bras J, **Beule D**, Dirnagl
U, Guerreiro R, Sassi C (2022). “TREX1 p.A129fs and p.Y305C variants in a large multi-ethnic cohort of CADASIL-like unrelated patients.”
doi:10.1016/j.neurobiolaging.2022.11.013  [☞ Link](https://doi.org/10.1016/j.neurobiolaging.2022.11.013)

Boschann F, Moreno D, Mensah M, Sczakiel H, Skipalova K, **Holtgrewe M**, Mundlos S, Fischer-Zirnsak B (2022). “Xq27.1 palindrome mediated interchromosomal
insertion likely causes familial congenital bilateral laryngeal abductor paralysis (Plott syndrome).” _J Hum Genet_, *67*(7), 405-410.
doi:10.1038/s10038-022-01018-z  [☞ Link](https://doi.org/10.1038/s10038-022-01018-z)

Seidel F, Laser K, Klingel K, Dartsch J, Theisen S, Pickardt T, **Holtgrewe M**, Gärtner A, Berger F, **Beule D**, Milting H, Schubert S, Klaassen S, Kühnisch J
(2022). “Pathogenic Variants in Cardiomyopathy Disorder Genes Underlie Pediatric Myocarditis-Further Impact of Heterozygous Immune Disorder Gene
Variants?” _J Cardiovasc Dev Dis_, *9*(7), 216. doi:10.3390/jcdd9070216  [☞ Link](https://doi.org/10.3390/jcdd9070216)

May V, Koch L, Fischer-Zirnsak B, Horn D, Gehle P, Kornak U, **Beule D**, **Holtgrewe M** (2022). “ClearCNV: CNV calling from NGS panel data in the presence of
ambiguity and noise.” _Bioinformatics_, *38*(16), 3871-3876. doi:10.1093/bioinformatics/btac418  [☞ Link](https://doi.org/10.1093/bioinformatics/btac418)

Schöpflin R, Melo U, Moeinzadeh H, Heller D, Laupert V, Hertzberg J, **Holtgrewe M**, Alavi N, Klever M, Jungnitsch J, Comak E, Türkmen S, Horn D, Duffourd
Y, Faivre L, Callier P, Sanlaville D, Zuffardi O, Tenconi R, Kurtas N, Giglio S, Prager B, Latos-Bielenska A, Vogel I, Bugge M, Tommerup N, Spielmann M,
Vitobello A, Kalscheuer V, Vingron M, Mundlos S (2022). “Integration of Hi-C with short and long-read genome sequencing reveals the structure of
germline rearranged genomes.” _Nat Commun_, *13*(1), 6470. doi:10.1038/s41467-022-34053-7  [☞ Link](https://doi.org/10.1038/s41467-022-34053-7)

Rieke D, de BT, Horak P, Lamping M, **Benary M**, Jelas I, Rüter G, Berger J, Zettwitz M, Kagelmann N, Kind A, Fabian F, **Beule D**, Glimm H, Brors B,
Stenzinger A, Fröhling S, Keilholz U (2022). “Feasibility and outcome of reproducible clinical interpretation of high-dimensional molecular data: a
comparison of two molecular tumor boards.” _BMC Med_, *20*, 367. ISSN 17417015, doi:10.1186/s12916-022-02560-5
 [☞ Link](https://doi.org/10.1186/s12916-022-02560-5)

Hinze C, Kocks C, Leiz J, Karaiskos N, Boltengagen A, Cao S, Skopnik C, Klocke J, Hardenberg J, Stockmann H, Gotthardt I, **Obermayer B**, Haghverdi L,
Wyler E, Landthaler M, Bachmann S, Hocke A, Corman V, Busch J, Schneider W, Himmerkus N, Bleich M, Eckardt K, Enghard P, Rajewsky N, Schmidt-Ott K
(2022). “Single-cell transcriptomics reveals common epithelial response patterns in human acute kidney injury.” _Genome Med_, *14*, 103.
doi:10.1186/s13073-022-01108-9  [☞ Link](https://doi.org/10.1186/s13073-022-01108-9)

Richter G, Wagner G, Reichenmiller K, Staufenbiel I, Martins O, Löscher B, **Holtgrewe M**, Jepsen S, Dommisch H, Schaefer A (2022). “Exome Sequencing of 5
Families with Severe Early-Onset Periodontitis.” _J Dent Res_, *101*(2), 151-157. doi:10.1177/00220345211029266
 [☞ Link](https://doi.org/10.1177/00220345211029266)

Coutelier M, **Holtgrewe M**, Jäger M, Flöttman R, Mensah M, Spielmann M, Krawitz P, Horn D, **Beule D**, Mundlos S (2022). “Combining callers improves the
detection of copy number variants from whole-genome sequencing.” _Eur J Hum Genet_, *30*(2), 178-186. doi:10.1038/s41431-021-00983-x
 [☞ Link](https://doi.org/10.1038/s41431-021-00983-x)

Vogt G, Verheyen S, Schwartzmann S, Ehmke N, Potratz C, Schwerin-Nagel A, Plecko B, **Holtgrewe M**, Seelow D, Blatterer J, Speicher M, Kornak U, Horn D,
Mundlos S, Fischer-Zirnsak B, Boschann F (2022). “Biallelic truncating variants in ATP9A cause a novel neurodevelopmental disorder involving postnatal
microcephaly and failure to thrive.” _J Med Genet_, *59*(7), 662-668. doi:10.1136/jmedgenet-2021-107843  [☞ Link](https://doi.org/10.1136/jmedgenet-2021-107843)
