Schinke C, Fernandez VV, **Ivanov  A**, Peng Y, Körtvelyessy P, Nolte L,
Huehnchen P, **Beule  D**, Stachelscheid H, Boehmerle W, Endres M (2021).
“Dataset for: Modeling chemotherapy induced neurotoxicity with human
induced pluripotent stem cell (iPSC)-derived sensory neurons.” _Data
Brief_, *38*, 107320. doi:10.1016/j.dib.2021.107320
 [☞ Link](https://doi.org/10.1016/j.dib.2021.107320)

**Weiner  J**, Suwalski P, **Holtgrewe  M**, Rakitko A, Thibeault C, Müller M,
Patriki D, Quedenau C, Krüger U, Ilinsky V, Popov I, Balnis J,
Jaitovich A, Helbig E, Lippert L, Stubbemann P, Real L, Macías J,
Pineda J, Fernandez-Fuertes M, Wang X, Karadeniz Z, Saccomanno J, Doehn
J, Hübner R, Hinzmann B, Salvo M, Blueher A, Siemann S, Jurisic S, Beer
J, Rutishauser J, Wiggli B, Schmid H, Danninger K, Binder R, Corman V,
Mühlemann B, Arjun AR, Fragiadakis G, Mick E, Comet C, Calfee C, Erle
D, Hendrickson C, Kangelaris K, Krummel M, Woodruff P, Langelier C,
Venkataramani U, García F, Zyla J, Drosten C, Alice B, Jones T, Suttorp
N, Witzenrath M, Hippenstiel S, Zemojtel T, Skurk C, Poller W, Borodina
T, Pa-Covid S, Ripke S, Sander L, **Beule  D**, Landmesser U, Guettouche T,
Kurth F, Heidecker B (2021). “Increased risk of severe clinical course
of COVID-19 in carriers of HLA-C\*04:01.” _EClinicalMedicine_, *40*,
101099. doi:10.1016/j.eclinm.2021.101099
 [☞ Link](https://doi.org/10.1016/j.eclinm.2021.101099)

Domaszewska T, Zyla J, Otto R, Kaufmann S, **Weiner  J** (2021). “Gene Set
Enrichment Analysis Reveals Individual Variability in Host Responses in
Tuberculosis Patients.” _Front Immunol_, *12*, 694680.
doi:10.3389/fimmu.2021.694680
 [☞ Link](https://doi.org/10.3389/fimmu.2021.694680)

Nouailles G, Wyler E, Pennitz P, Postmus D, Vladimirova D, Kazmierski
J, Pott F, Dietert K, Muelleder M, Farztdinov V, **Obermayer  B**, Wienhold
S, Andreotti S, Hoefler T, Sawitzki B, Drosten C, Sander LE, Suttorp N,
Ralser M, **Beule  D**, Gruber AD, Goffinet C, Landthaler M, Trimpert J,
Witzenrath M (2021). “Single-cell-sequencing in SARS-COV-2-infected
hamsters sheds light on endothelial cell involvement in COVID-19.”
_European Respiratory Journal_.
doi:10.1183/13993003.congress-2021.PA2355
 [☞ Link](https://erj.ersjournals.com/content/58/suppl_65/PA2355.abstract)

Bischoff P, Trinks A, **Obermayer  B**, **Pett  J**, Wiederspahn J, Uhlitz F,
Liang X, Lehmann A, Jurmeister P, Elsner A, Dziodzio T, Rückert J,
Neudecker J, Falk C, **Beule  D**, Sers C, Morkel M, Horst D, Blüthgen N,
Klauschen F (2021). “Single-cell RNA sequencing reveals distinct tumor
microenvironmental patterns in lung adenocarcinoma.” _Oncogene_, *40*,
6748-6758. doi:10.1038/s41388-021-02054-3
 [☞ Link](https://doi.org/10.1038/s41388-021-02054-3)

Uhlitz F, Bischoff P, Peidli S, Sieber A, Trinks A, Lüthen M, Obermayer
B, **Blanc  E**, Ruchiy Y, Sell T, Mamlouk S, Arsie R, Wei T, Klotz-Noack K,
Schwarz R, Sawitzki B, Kamphues C, **Beule  D**, Landthaler M, Sers C, Horst
D, Blüthgen N, Morkel M (2021). “Mitogen-activated protein kinase
activity drives cell trajectories in colorectal cancer.” _EMBO Mol
Med_, *13*, e14123. doi:10.15252/emmm.202114123
 [☞ Link](https://doi.org/10.15252/emmm.202114123)

Nouailles G, Wyler E, Pennitz P, Postmus D, Vladimirova D, Kazmierski
J, Pott F, Dietert K, Muelleder M, Farztdinov V, **Obermayer  B**, Wienhold
S, Andreotti S, Hoefler T, Sawitzki B, Drosten C, Sander L, Suttorp N,
Ralser M, **Beule  D**, Gruber A, Goffinet C, Landthaler M, Trimpert J,
Witzenrath M (2021). “Temporal omics analysis in Syrian hamsters
unravel cellular effector responses to moderate COVID-19.” _Nat
Commun_, *12*, 4869. doi:10.1038/s41467-021-25030-7
 [☞ Link](https://doi.org/10.1038/s41467-021-25030-7)

Paterson S, Kar S, Ung S, Gardener Z, Bergstrom E, Ascough S, Kalyan M,
Zyla J, Maertzdorf J, Mollenkopf H, **Weiner  J**, Jozwik A, Jarvis H, Jha
A, Nicholson B, Veldman T, Woods C, Mallia P, Kon O, Kaufmann S,
Openshaw P, Chiu C (2021). “Innate-like Gene Expression of
Lung-Resident Memory CD8(+) T Cells during Experimental Human
Influenza: A Clinical Study.” _Am J Respir Crit Care Med_, *204*,
826-841. doi:10.1164/rccm.202103-0620OC
 [☞ Link](https://doi.org/10.1164/rccm.202103-0620OC)

Rajasekaran S, Siddiqui J, Rakijas J, Nicolay B, Lin C, Khan E, Patel
R, Morris R, Wyler E, Boukhali M, Balasubramanyam J, Ranjith KR, van
Rechem C, Vogel C, Elchuri S, Landthaler M, **Obermayer  B**, Haas W, Dyson
N, Miles W (2021). “Integrated multi-omics analysis of RB-loss
identifies widespread cellular programming and synthetic weaknesses.”
_Commun Biol_, *4*, 977. doi:10.1038/s42003-021-02495-2
 [☞ Link](https://doi.org/10.1038/s42003-021-02495-2)

Schinke C, Fernandez VV, **Ivanov  A**, Peng Y, Körtvelyessy P, Nolte L,
Huehnchen P, **Beule  D**, Stachelscheid H, Boehmerle W, Endres M (2021).
“Modeling chemotherapy induced neurotoxicity with human induced
pluripotent stem cell (iPSC) -derived sensory neurons.” _Neurobiol
Dis_, *155*, 105391. doi:10.1016/j.nbd.2021.105391
 [☞ Link](https://doi.org/10.1016/j.nbd.2021.105391)

Pei G, Zyla J, He L, Moura-Alves P, Steinle H, Saikali P, Lozza L,
Nieuwenhuizen N, **Weiner  J**, Mollenkopf H, Ellwanger K, Arnold C, Duan M,
Dagil Y, Pashenkov M, Boneca I, Kufer T, Dorhoi A, Kaufmann S (2021).
“Cellular stress promotes NOD1/2-dependent inflammation via the
endogenous metabolite sphingosine-1-phosphate.” _EMBO J_, *40*,
e106272. doi:10.15252/embj.2020106272
 [☞ Link](https://doi.org/10.15252/embj.2020106272)

Herda S, Heimann A, **Obermayer  B**, Ciraolo E, Althoff S, Ruß J, Grunert
C, Busse A, Bullinger L, Pezzutto A, Blankenstein T, **Beule  D**, Na I
(2021). “Long-term in vitro expansion ensures increased yield of
central memory T cells as perspective for manufacturing challenges.”
_Int J Cancer_, *148*, 3097-3110. doi:10.1002/ijc.33523
 [☞ Link](https://doi.org/10.1002/ijc.33523)

Kraushar M, Krupp F, Harnett D, Turko P, Ambrozkiewicz M, Sprink T,
Imami K, Günnigmann M, Zinnall U, Vieira-Vieira C, Schaub T,
Münster-Wandowski A, Bürger J, Borisova E, Yamamoto H, Rasin M, Ohler
U, **Beule  D**, Mielke T, Tarabykin V, Landthaler M, Kramer G, Vida I,
Selbach M, Spahn C (2021). “Protein Synthesis in the Developing
Neocortex at Near-Atomic Resolution Reveals Ebp1-Mediated Neuronal
Proteostasis at the 60S Tunnel Exit.” _Mol Cell_, *81*, 304-322.e16.
doi:10.1016/j.molcel.2020.11.037
 [☞ Link](https://doi.org/10.1016/j.molcel.2020.11.037)

Heyckendorf J, Marwitz S, Reimann M, Avsar K, DiNardo A, Günther G,
Hoelscher M, Ibraim E, Kalsdorf B, Kaufmann S, Kontsevaya I, van Leth
F, Mandalakas A, Maurer F, Müller M, Nitschkowski D, Olaru I, Popa C,
Rachow A, Rolling T, Rybniker J, Salzer H, Sanchez-Carballo P,
Schuhmann M, Schaub D, Spinu V, Suárez I, Terhalle E, Unnewehr M,
**Weiner  J3**, Goldmann T, Lange C (2021). “Prediction of anti-tuberculosis
treatment duration based on a 22-gene transcriptomic model.” _Eur
Respir J_, *58*. doi:10.1183/13993003.03492-2020
 [☞ Link](https://doi.org/10.1183/13993003.03492-2020)

Bao X, **Weiner  J3**, Meckes O, Dommisch H, Schaefer A (2021). “Entamoeba
gingivalis Exerts Severe Pathogenic Effects on the Oral Mucosa.” _J
Dent Res_, *100*, 771-776. doi:10.1177/00220345211004498
 [☞ Link](https://doi.org/10.1177/00220345211004498)

Eckold C, Kumar V, **Weiner  J**, Alisjahbana B, Riza A, Ronacher K, Coronel
J, Kerry-Barnard S, Malherbe S, Kleynhans L, Stanley K, Ruslami R,
Ioana M, Ugarte-Gil C, Walzl G, van Crevel R, Wijmenga C, Critchley J,
Dockrell H, Cliff J, TANDEM c (2021). “Impact of Intermediate
Hyperglycemia and Diabetes on Immune Dysfunction in Tuberculosis.”
_Clin Infect Dis_, *72*, 69-78. doi:10.1093/cid/ciaa751
 [☞ Link](https://doi.org/10.1093/cid/ciaa751)

Wyler E, Mösbauer K, Franke V, Diag A, Gottula L, Arsiè R, Klironomos
F, Koppstein D, Hönzke K, Ayoub S, Buccitelli C, Hoffmann K, Richter A,
Legnini I, **Ivanov  A**, Mari T, Del GS, Papies J, Praktiknjo S, Meyer T,
Müller M, Niemeyer D, Hocke A, Selbach M, Akalin A, Rajewsky N, Drosten
C, Landthaler M (2021). “Transcriptomic profiling of SARS-CoV-2
infected human cell lines identifies HSP90 as target for COVID-19
therapy.” _iScience_, *24*, 102151. doi:10.1016/j.isci.2021.102151
 [☞ Link](https://doi.org/10.1016/j.isci.2021.102151)

Mölder F, Jablonski K, Letcher B, Hall M, Tomkins-Tinch C, Sochat V,
Forster J, Lee S, Twardziok S, Kanitz A, Wilm A, **Holtgrewe  M**, Rahmann
S, Nahnsen S, Köster J (2021). “Sustainable data analysis with
Snakemake.” _F1000Res_, *10*, 33. doi:10.12688/f1000research.29032.2
 [☞ Link](https://doi.org/10.12688/f1000research.29032.2)

**Messerschmidt  C**, Foddis M, Blumenau S, Müller S, **Bentele  K**, Holtgrewe
M, Kun-Rodrigues C, Alonso I, do CMM, Morgadinho A, Velon A, Santo G,
Santana I, Mönkäre S, Kuuluvainen L, Schleutker J, Pöyhönen M,
Myllykangas L, Senatore A, Berchtold D, Winek K, Meisel A, Pavlovic A,
Kostic V, Dobricic V, Lohmann E, Hanagasi H, Guven G, Bilgic B, Bras J,
Guerreiro R, **Beule  D**, Dirnagl U, Sassi C (2021). “PHACTR1 genetic
variability is not critical in small vessel ischemic disease patients
and PcomA recruitment in C57BL/6J mice.” _Sci Rep_, *11*(1), 6072.
doi:10.1038/s41598-021-84919-x
 [☞ Link](https://doi.org/10.1038/s41598-021-84919-x)

Zarate Y, Uehara T, Abe K, Oginuma M, Harako S, Ishitani S, Lehesjoki
A, Bierhals T, Kloth K, Ehmke N, Horn D, **Holtgrewe  M**, Anderson K,
Viskochil D, Edgar-Zarate C, Sacoto M, Schnur R, Morrow M,
Sanchez-Valle A, Pappas J, Rabin R, Muona M, Anttonen A, Platzer K,
Luppe J, Gburek-Augustat J, Kaname T, Okamoto N, Mizuno S, Kaido Y,
Ohkuma Y, Hirose Y, Ishitani T, Kosaki K (2021). “CDK19-related
disorder results from both loss-of-function and gain-of-function de
novo missense variants.” _Genet Med_, *23*(6), 1050-1057.
doi:10.1038/s41436-020-01091-9
 [☞ Link](https://doi.org/10.1038/s41436-020-01091-9)

Vogt G, El CN, Herczegfalvi Á, Kölbel H, Lekaj A, Schara U, Holtgrewe
M, Krause S, Horvath R, Schuelke M, Hübner C, Mundlos S, Roos A,
Lochmüller H, Karcagi V, Kornak U, Fischer-Zirnsak B (2021). “Expanding
the clinical and molecular spectrum of ATP6V1A related metabolic cutis
laxa.” _J Inherit Metab Dis_, *44*(4), 972-986. doi:10.1002/jimd.12341
 [☞ Link](https://doi.org/10.1002/jimd.12341)

Yildirim F, Foddis M, Blumenau S, Müller S, Kajetan B, **Holtgrewe  M**,
Kola V, **Beule  D**, Sassi C (2021). “Shared and oppositely regulated
transcriptomic signatures in Huntington's disease and brain ischemia
confirm known and unveil novel potential neuroprotective genes.”
_Neurobiol Aging_, *104*, 122.e1-122.e17.
doi:10.1016/j.neurobiolaging.2021.03.001
 [☞ Link](https://doi.org/10.1016/j.neurobiolaging.2021.03.001)

Elsner J, Mensah M, **Holtgrewe  M**, Hertzberg J, Bigoni S, Busche A,
Coutelier M, de SD, Elçioglu N, Filges I, Gerkes E, Girisha K,
Graul-Neumann L, Jamsheer A, Krawitz P, Kurth I, Markus S, Megarbane A,
Reis A, Reuter M, Svoboda D, Teller C, Tuysuz B, Türkmen S, Wilson M,
Woitschach R, Vater I, Caliebe A, Hülsemann W, Horn D, Mundlos S,
Spielmann M (2021). “Genome sequencing in families with congenital limb
malformations.” _Hum Genet_, *140*(8), 1229-1239.
doi:10.1007/s00439-021-02295-y
 [☞ Link](https://doi.org/10.1007/s00439-021-02295-y)

Seidel F, **Holtgrewe  M**, Al-Wakeel-Marquard N, Opgen-Rhein B, Dartsch J,
Herbst C, **Beule  D**, Pickardt T, Klingel K, Messroghli D, Berger F,
Schubert S, Kühnisch J, Klaassen S (2021). “Pathogenic Variants
Associated With Dilated Cardiomyopathy Predict Outcome in Pediatric
Myocarditis.” _Circ Genom Precis Med_, *14*(4), e003250.
doi:10.1161/CIRCGEN.120.003250
 [☞ Link](https://doi.org/10.1161/CIRCGEN.120.003250)

Melo U, Piard J, Fischer-Zirnsak B, Klever M, Schöpflin R, Mensah M,
**Holtgrewe  M**, Arbez-Gindre F, Martin A, Guigue V, Gaillard D, Landais E,
Roze V, Kremer V, Ramanah R, Cabrol C, Harms F, Kornak U, Spielmann M,
Mundlos S, van Maldergem L (2021). “Complete lung agenesis caused by
complex genomic rearrangements with neo-TAD formation at the SHH
locus.” _Hum Genet_, *140*(10), 1459-1469.
doi:10.1007/s00439-021-02344-6
 [☞ Link](https://doi.org/10.1007/s00439-021-02344-6)

Sczakiel H, Hülsemann W, **Holtgrewe  M**, Abad-Perez A, Elsner J,
Schwartzmann S, Horn D, Spielmann M, Mundlos S, Mensah M (2021). “GLI3
variants causing isolated polysyndactyly are not restricted to the
protein's C-terminal third.” _Clin Genet_, *100*(6), 758-765.
doi:10.1111/cge.14059  [☞ Link](https://doi.org/10.1111/cge.14059)
